<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>ClonalFrame</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />

</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 64px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 69px;
  margin-top: -69px;
}

.section h2 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h3 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h4 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h5 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h6 {
  padding-top: 69px;
  margin-top: -69px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Xavier Didelot</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="research.html">Research</a>
</li>
<li>
  <a href="software.html">Software</a>
</li>
<li>
  <a href="pubs.html">Publications</a>
</li>
<li>
  <a href="bio.html">Background</a>
</li>
<li>
  <a href="contact.html">Contact</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="contact.html">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/xavierdidelot">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/xavierdidelot">
    <span class="fa fa-twitter fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://www.linkedin.com/in/xavier-didelot-538bb531/">
    <span class="fa fa-linkedin fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://scholar.google.co.uk/citations?user=ouakQ4cAAAAJ">
    <span class="fa fa-graduation-cap fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">ClonalFrame</h1>

</div>


<div id="important-notice" class="section level2">
<h2>Important notice</h2>
<p>This webpage is about the original ClonalFrame released in 2007 and designed mostly to work on MLST data or small numbers of genomes. In 2015 a new separate software was released called ClonalFrameML which is designed to run on large whole genomes datasets. For more information about ClonalFrameML, see the separate <a href="https://github.com/xavierdidelot/ClonalFrameML">ClonalFrameML website</a>.</p>
</div>
<div id="introduction" class="section level2">
<h2>Introduction</h2>
<p>This is the homepage of ClonalFrame, a computer package for the inference of bacterial microevolution using multilocus sequence data.</p>
<p>In a nutshell, ClonalFrame identifies the clonal relationships between the members of a sample, while also estimating the chromosomal position of homologous recombination events that have disrupted the clonal inheritance.</p>
<p>ClonalFrame can be applied to any kind of sequence data, from a single fragment of DNA to whole genomes. It is well suited for the analysis of <a href="http://pubmlst.org/">MLST data</a>, where 7 gene fragments have been sequenced, but becomes progressively more powerful as the sequenced regions increase in length and number up to <a href="http://www.genomesonline.org/">whole genomes</a>. However, it requires the sequences to be aligned. If you have genomic data that is not aligned, we recommend using <a href="http://gel.ahabs.wisc.edu/mauve/">Mauve</a> which produces alignment of whole bacterial genomes in exactly the format required for analysis with ClonalFrame.</p>
<p>The methods used in ClonalFrame are presented in the paper “Inference of bacterial microevolution using multilocus sequence data” by <a href="http://www.genetics.org/cgi/content/abstract/175/3/1251">Didelot and Falush (2007)</a>, which is the appropriate citation for this program.</p>
</div>
<div id="getting-help" class="section level2">
<h2>Getting help</h2>
<p>Click here to download the <a href="http://www.stats.ox.ac.uk/~didelot/files/clonalframe-userguide.pdf">ClonalFrame user guide in PDF format</a>. It is also included in each downloadable package.</p>
<p>The paper describing the methods is also available: <a href="http://www.genetics.org/cgi/content/abstract/175/3/1251">Didelot and Falush (2007)</a></p>
<p>If you have a question or a problem that is not discussed in the user guide, please <a href="contact.htm">get in touch</a>.</p>
</div>
<div id="download" class="section level2">
<h2>Download</h2>
<div id="version-1.2-current-release" class="section level3">
<h3>Version 1.2 (current release):</h3>
<ul>
<li>Starting with this version, ClonalFrame is distributed under the terms of <a href="http://www.gnu.org/licenses/gpl.html">GNU General Public License</a>.</li>
<li><a href="http://www.stats.ox.ac.uk/~didelot/files/ClonalFrame-1.2.tar.gz">Click here</a> to download the source code of ClonalFrame. Installation is done either via the qmake utility using the command <code>qmake;make</code> or directly using the command <code>g++ -lgsl *.cpp -o ClonalFrame</code></li>
<li><a href="http://www.stats.ox.ac.uk/~didelot/files/ClonalFrame-1.2-Mac.zip">Click here</a> to download a precompiled binary for Mac.</li>
<li><a href="http://www.stats.ox.ac.uk/~didelot/files/ClonalFrame-1.2-Lin.zip">Click here</a> to download a precompiled binary for Linux.</li>
<li>For the GUI of ClonalFrame, please download the Windows version 1.1 below. Please note that this GUI is no longer supported and was originally designed to work for Windows XP. Some users have reported that it could be made to work for Windows 7/8 using the procedure described <a href="http://windows.microsoft.com/en-GB/windows7/products/features/windows-xp-mode">here</a>. The Windows GUI can also be used on a Mac via <a href=http://www.winehq.org/>Wine</a> or the <a href="http://winebottler.kronenberg.org/">WineBottler</a>.</li>
</ul>
</div>
<div id="version-1.1" class="section level3">
<h3>Version 1.1:</h3>
<ul>
<li><a href="http://www.stats.ox.ac.uk/~didelot/files/clonalframeinstaller.zip">Click here</a> to download ClonalFrame and GUI for Windows.</li>
<li><a href="http://www.stats.ox.ac.uk/~didelot/files/clonalframelinuxgui.zip">Click here</a> to download ClonalFrame and GUI for Linux.</li>
<li><a href="http://www.stats.ox.ac.uk/~didelot/files/clonalframewin.zip">Click here</a> to download ClonalFrame for Windows without the GUI.</li>
<li><a href="http://www.stats.ox.ac.uk/~didelot/files/clonalframelinux.zip">Click here</a> to download ClonalFrame for Linux without the GUI.</li>
<li><a href="http://www.stats.ox.ac.uk/~didelot/files/clonalframelinux64.zip">Click here</a> to download ClonalFrame for 64bit/Linux without the GUI.</li>
<li><a href="http://www.stats.ox.ac.uk/~didelot/files/clonalframemac.zip">Click here</a> to download ClonalFrame for Mac.</li>
</ul>
</div>
<div id="version-1.0" class="section level3">
<h3>Version 1.0:</h3>
<ul>
<li><a href="http://www.stats.ox.ac.uk/~didelot/files/clonalframeinstaller10.zip">Click here</a> to download ClonalFrame and GUI version 1.0 for Windows.</li>
</ul>
</div>
</div>
<div id="xmfa2struct" class="section level2">
<h2>xmfa2struct</h2>
<p>xmfa2struct is a program written in C which converts files in eXtended Multi-Fasta format (XMFA, the input format of ClonalFrame) into the input file format of <a href="http://pritch.bsd.uchicago.edu/structure.html">Structure</a>.</p>
<ul>
<li><a href="http://www.stats.ox.ac.uk/~didelot/files/xmfa2struct.c">Click here</a> to download the source code of xmfa2struct</li>
<li><a href="http://www.stats.ox.ac.uk/~didelot/files/xmfa2struct.zip">Click here</a> to download a Windows executable</li>
<li><a href="http://www.stats.ox.ac.uk/~didelot/files/xmfa2struct.pdf">Click here</a> to download the xmfa2struct readme file in PDF format</li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
