<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Publications</title>

<script src="site_libs/header-attrs-2.28/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cerulean.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Home</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="research.html">Research</a>
</li>
<li>
  <a href="software.html">Software</a>
</li>
<li>
  <a href="pubs.html">Publications</a>
</li>
<li>
  <a href="bio.html">Background</a>
</li>
<li>
  <a href="contact.html">Contact</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="contact.html">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/xavierdidelot">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/xavierdidelot">
    <span class="fa fa-twitter fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://www.linkedin.com/in/xavier-didelot-538bb531/">
    <span class="fa fa-linkedin fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://scholar.google.co.uk/citations?user=ouakQ4cAAAAJ">
    <span class="fa fa-graduation-cap fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Publications</h1>

</div>


<div id="preprints" class="section level3">
<h3>Preprints</h3>
<ol start="200">
<li>
<p>
Adeoye M, Didelot X, Spencer SE (9999).“Bayesian Spatio-Temporal
Modelling for Infectious Disease Outbreak Detection.”<em>arXiv</em>,
10.48550/arXiv.2503.01456.<a href="https://doi.org/10.48550/arXiv.2503.01456">doi:10.48550/arXiv.2503.01456</a>,
2503.01456.
</p>
</li>
</ol>
<ol start="199">
<li>
<p>
Yang C, Huang H, Wang N, Didelot X, Yang R, Cui Y, Falush D
(9999).“Macrogenetic Atlas of Prokaryotes.”<em>bioRxiv</em>,
10.1101/2025.02.26.640246.<a href="https://doi.org/10.1101/2025.02.26.640246">doi:10.1101/2025.02.26.640246</a>.
</p>
</li>
</ol>
<ol start="198">
<li>
<p>
Craddock H, Spencer SE, Didelot X (9999).“A Bayesian Modelling Framework
with Model Comparison for Epidemics with
Super-Spreading.”<em>arXiv</em>,
10.48550/arXiv.2501.12768.<a href="https://doi.org/10.48550/arXiv.2501.12768">doi:10.48550/arXiv.2501.12768</a>,
2501.12768.
</p>
</li>
</ol>
<ol start="197">
<li>
<p>
Gill A, Koskela J, Didelot X, Everitt RG (9999).“Bayesian Inference of
Reproduction Number from Epidemiological and Genetic Data Using Particle
MCMC.”<em>arXiv</em>,
10.48550/arXiv.2311.09838.<a href="https://doi.org/10.48550/arXiv.2311.09838">doi:10.48550/arXiv.2311.09838</a>.
</p>
</li>
</ol>
<ol start="196">
<li>
<p>
Karah N, Faille N, Grenier F, Abou-Fayad A, Higgins PG, Al-Hassan L,
Evans BA, Poirel L, Bonnin R, Hammerum AM, Hansen F, Rafei R, Hamze M,
Didelot X, Castillo-Ramírez S, Lévesque S, Rodrigue S, Uhlin BE, Haraoui
L (9999).“Emergence of <em>Acinetobacter</em><em> Baumannii</em>
International Clone 10 Predominantly Found in the Middle
East.”<em>bioRxiv</em>,
10.1101/2023.10.09.561570.<a href="https://doi.org/10.1101/2023.10.09.561570">doi:10.1101/2023.10.09.561570</a>.
</p>
</li>
</ol>
<ol start="195">
<li>
<p>
Didelot X, Ribeca P (9999).“KPop : An Assembly-Free and Scalable Method
for the Comparative Analysis of Microbial Genomes.”<em>bioRxiv</em>,
2022.06.22.497172.<a href="https://doi.org/10.1101/2022.06.22.497172">doi:10.1101/2022.06.22.497172</a>.
</p>
</li>
</ol>
</div>
<div id="section" class="section level3">
<h3>2025</h3>
<ol start="194">
<li>
<p>
Didelot X, Helekal D, Roberts I (2025).“Ancestral Process for Infectious
Disease Outbreaks with Superspreading.”<em>Journal of Theoretical
Biology</em>, <b>607</b>,
112109.<a href="https://doi.org/10.1016/j.jtbi.2025.112109">doi:10.1016/j.jtbi.2025.112109</a>.
</p>
</li>
</ol>
<ol start="193">
<li>
<p>
Helekal D, Koskela J, Didelot X (2025).“Inference of Multiple Mergers
While Dating a Pathogen Phylogeny.”<em>Systematic Biology</em>,
syaf003.<a href="https://doi.org/10.1093/sysbio/syaf003">doi:10.1093/sysbio/syaf003</a>.
</p>
</li>
</ol>
<ol start="192">
<li>
<p>
Yang Z, Davies H, Clapham J, Lopez L, Bratcher HB, Tiong A, Didelot X,
Maiden MC, Carter PE, Ren X (2025).“Phylodynamic Analysis of a Prolonged
Meningococcal Epidemic Reveals Multiple Introductions and Pre-Epidemic
Expansion.”<em>Infection, Genetics and Evolution</em>,
105726.<a href="https://doi.org/10.1016/j.meegid.2025.105726">doi:10.1016/j.meegid.2025.105726</a>.
</p>
</li>
</ol>
<ol start="191">
<li>
<p>
Roberts I, Everitt RG, Koskela J, Didelot X (2025).“Bayesian Inference
of Pathogen Phylogeography Using the Structured Coalescent
Model.”<em>PLOS Computational Biology</em>, <b>21</b>(4),
e1012995.<a href="https://doi.org/10.1371/journal.pcbi.1012995">doi:10.1371/journal.pcbi.1012995</a>.
</p>
</li>
</ol>
<ol start="190">
<li>
<p>
Carson J, Keeling M, Ribeca P, Didelot X (2025).“Incorporating
Epidemiological Data into the Genomic Analysis of Partially Sampled
Infectious Disease Outbreaks.”<em>Molecular Biology and Evolution</em>,
<b>42</b>(4),
msaf083.<a href="https://doi.org/10.1093/molbev/msaf083">doi:10.1093/molbev/msaf083</a>.
</p>
</li>
</ol>
<ol start="189">
<li>
<p>
Hoefer A, Seth-Smith H, Palma F, Schindler S, Freschi L, Dangel A,
Berger A, D’Aeth J, Cordery R, Delgado-Rodriguez E, Gruner E, Flury D,
Hinic V, Kofler J, Lienhard R, Mariman R, Nolte O, Schibli A, Toubiana
J, Traugott M, Jacquinet S, Indra A, Fry NK, Palm D, Sing A, Brisse S,
Egli A (2025).“Corynebacterium Diphtheriae Outbreak in Migrant
Populations in Europe.”<em>N Engl J Med</em>,
NEJMoa2311981.<a href="https://doi.org/10.1056/NEJMoa2311981">doi:10.1056/NEJMoa2311981</a>.
</p>
</li>
</ol>
</div>
<div id="section-1" class="section level3">
<h3>2024</h3>
<ol start="188">
<li>
<p>
Carson J, Keeling M, Wyllie D, Ribeca P, Didelot X (2024).“Inference of
Infectious Disease Transmission through a Relaxed Bottleneck Using
Multiple Genomes per Host.”<em>Molecular Biology and Evolution</em>,
<b>41</b>,
msad288.<a href="https://doi.org/10.1093/molbev/msad288">doi:10.1093/molbev/msad288</a>.
</p>
</li>
</ol>
<ol start="187">
<li>
<p>
Fotopoulou ET, Jenkins C, Barker CR, Painset A, Didelot X, Simbo A,
Douglas A, Godbole G, Jorgensen F, Gharbia S, McLauchlin J
(2024).“Genomic Epidemiology of the Clinically Dominant Clonal Complex 1
in the Listeria Monocytogenes Population in the UK.”<em>Microbial
Genomics</em>, <b>10</b>,
001155.<a href="https://doi.org/10.1099/mgen.0.001155">doi:10.1099/mgen.0.001155</a>.
</p>
</li>
</ol>
<ol start="186">
<li>
<p>
Wan Y, Myall AC, Boonyasiri A, Bolt F, Ledda A, Mookerjee S, Weiße AY,
Getino M, Turton JF, Abbas H, Prakapaite R, Sabnis A, Abdolrasouli A,
Malpartida-Cardenas K, Miglietta L, Donaldson H, Gilchrist M, Hopkins
KL, Ellington MJ, Otter JA, Larrouy-Maumus G, Edwards AM,
Rodriguez-Manzano J, Didelot X, Barahona M, Holmes AH, Jauneikaite E,
Davies F (2024).“Integrated Analysis of Patient Networks and Plasmid
Genomes Reveals a Regional, Multi-Species Outbreak of
Carbapenemase-Producing Enterobacterales Carrying Both blaIMP and Mcr-9
Genes.”<em>The Journal of Infectious Diseases</em>, <b>230</b>,
e159–e170.<a href="https://doi.org/10.1093/infdis/jiae019">doi:10.1093/infdis/jiae019</a>.
</p>
</li>
</ol>
<ol start="185">
<li>
<p>
Nimmo C, Ortiz AT, Tan CCS, Pang J, Acman M, Millard J, Padayatchi N,
Grant AD, O’Donnell M, Pym A, Brynildsrud OB, Eldholm V, Grandjean L,
Didelot X, Balloux F, vanDorp L (2024).“Detection of a Historic
Reservoir of Bedaquiline/Clofazimine Resistance-Associated Variants in
Mycobacterium Tuberculosis.”<em>Genome Medicine</em>, <b>16</b>(1),
34.<a href="https://doi.org/10.1186/s13073-024-01289-5">doi:10.1186/s13073-024-01289-5</a>.
</p>
</li>
</ol>
<ol start="184">
<li>
<p>
Taylor AJ, Yahara K, Pascoe B, Ko S, Mageiros L, Mourkas E, Calland JK,
Puranen S, Hitchings MD, Jolley KA, Kobras CM, Bayliss S, Williams NJ,
Van Vliet AHM, Parkhill J, Maiden MCJ, Corander J, Hurst LD, Falush D,
Keim P, Didelot X, Kelly DJ, Sheppard SK (2024).“Epistasis, Core-Genome
Disharmony, and Adaptation in Recombining Bacteria.”<em>mBio</em>,
<b>15</b>(6),
e00581-24.<a href="https://doi.org/10.1128/mbio.00581-24">doi:10.1128/mbio.00581-24</a>.
</p>
</li>
</ol>
<ol start="183">
<li>
<p>
Avila Cartes J, Bonizzoni P, Ciccolella S, Della Vedova G, Denti L,
Didelot X, Monti D, Pirola Y (2024).“RecGraph: Recombination-Aware
Alignment of Sequences to Variation Graphs.”<em>Bioinformatics</em>,
btae292.<a href="https://doi.org/10.1093/bioinformatics/btae292">doi:10.1093/bioinformatics/btae292</a>.
</p>
</li>
</ol>
<ol start="182">
<li>
<p>
Osman R, Dema E, David A, Hughes G, Field N, Cole M, Didelot X, Saunders
J (2024).“Understanding the Potential Role of Whole Genome Sequencing
(WGS) in Managing Patients with Gonorrhoea: A Systematic Review of WGS
Use on Human Pathogens in Individual Patient Care.”<em>Journal of
Infection</em>, <b>88</b>,
106168.<a href="https://doi.org/10.1016/j.jinf.2024.106168">doi:10.1016/j.jinf.2024.106168</a>.
</p>
</li>
</ol>
<ol start="181">
<li>
<p>
Moore MP, Laager M, Ribeca P, Didelot X (2024).“KmerAperture: Retaining
k-Mer Synteny for Alignment-Free Extraction of Core and Accessory
Differences between Bacterial Genomes.”<em>PLOS Genetics</em>,
<b>20</b>(4),
e1011184.<a href="https://doi.org/10.1371/journal.pgen.1011184">doi:10.1371/journal.pgen.1011184</a>.
</p>
</li>
</ol>
<ol start="180">
<li>
<p>
Nair S, Barker CR, Bird M, Greig DR, Collins C, Painset A, Chattaway M,
Pickard D, Larkin L, Gharbia S, Didelot X, Ribeca P (2024).“Presence of
Phage-Plasmids in Multiple Serovars of Salmonella
Enterica.”<em>Microbial Genomics</em>, <b>10</b>,
001247.<a href="https://doi.org/10.1099/mgen.0.001247">doi:10.1099/mgen.0.001247</a>.
</p>
</li>
</ol>
<ol start="179">
<li>
<p>
Vieira A, Wan Y, Ryan Y, Li HK, Guy RL, Papangeli M, Huse KK, Reeves LC,
Soo VWC, Daniel R, Harley A, Broughton K, Dhami C, Ganner M, Ganner MA,
Mumin Z, Razaei M, Rundberg E, Mammadov R, Mills EA, Sgro V, Mok KY,
Didelot X, Croucher NJ, Jauneikaite E, Lamagni T, Brown CS, Coelho J,
Sriskandan S (2024).“Rapid Expansion and International Spread of M1UK in
the Post-Pandemic UK Upsurge of Streptococcus Pyogenes.”<em>Nature
Communications</em>, <b>15</b>(1),
3916.<a href="https://doi.org/10.1038/s41467-024-47929-7">doi:10.1038/s41467-024-47929-7</a>.
</p>
</li>
</ol>
<ol start="178">
<li>
<p>
Kendall M, Ferretti L, Wymant C, Tsallis D, Petrie J, Di Francia A, Di
Lauro F, Abeler-Dörner L, Manley H, Panovska-Griffiths J, Ledda A,
Didelot X, Fraser C (2024).“Drivers of Epidemic Dynamics in Real Time
from Daily Digital COVID-19 Measurements.”<em>Science</em>, <b>385</b>,
eadm8103.<a href="https://doi.org/10.1126/science.adm8103">doi:10.1126/science.adm8103</a>.
</p>
</li>
</ol>
<ol start="177">
<li>
<p>
Vale FF, HpGP Research Network, Roberts RJ, Kobayashi I, Camargo MC,
Rabkin CS (2024).“Gene Content, Phage Cycle Regulation Model and
Prophage Inactivation Disclosed by Prophage Genomics in the
<em>Helicobacter</em><em> Pylori</em> Genome Project.”<em>Gut
Microbes</em>, <b>16</b>(1),
2379440.<a href="https://doi.org/10.1080/19490976.2024.2379440">doi:10.1080/19490976.2024.2379440</a>.
</p>
</li>
</ol>
</div>
<div id="section-2" class="section level3">
<h3>2023</h3>
<ol start="176">
<li>
<p>
Didelot X, Helekal D, Kendall M, Ribeca P (2023).“Distinguishing
Imported Cases from Locally Acquired Cases within a Geographically
Limited Genomic Sample of an Infectious
Disease.”<em>Bioinformatics</em>, <b>23</b>,
btac761.<a href="https://doi.org/10.1093/bioinformatics/btac761">doi:10.1093/bioinformatics/btac761</a>.
</p>
</li>
</ol>
<ol start="175">
<li>
<p>
Didelot X, Franceschi V, Frost SDW, Dennis A, Volz EM (2023).“Model
Design for Non-Parametric Phylodynamic Inference and Applications to
Pathogen Surveillance.”<em>Virus Evolution</em>, <b>9</b>,
vead028.<a href="https://doi.org/10.1093/ve/vead028">doi:10.1093/ve/vead028</a>.
</p>
</li>
</ol>
<ol start="174">
<li>
<p>
Didelot X (2023).“Phylogenetic Analysis of Bacterial Pathogen
Genomes.”In Nordenfelt P, Collin M (eds.), <em>Bacterial Pathogenesis:
Methods and Protocols</em>, 87–99.Springer US, New York, NY.ISBN
978-1-0716-3243-7,
<a href="https://doi.org/10.1007/978-1-0716-3243-7_6">doi:10.1007/978-1-0716-3243-7_6</a>.
</p>
</li>
</ol>
<ol start="173">
<li>
<p>
Helekal D, Keeling M, Grad YH, Didelot X (2023).“Estimating the Fitness
Cost and Benefit of Antimicrobial Resistance from Pathogen Genomic
Data.”<em>Journal of The Royal Society Interface</em>, <b>20</b>,
20230074.<a href="https://doi.org/10.1098/rsif.2023.0074">doi:10.1098/rsif.2023.0074</a>.
</p>
</li>
</ol>
<ol start="172">
<li>
<p>
Kendall M, Tsallis D, Wymant C, Di Francia A, Balogun Y, Didelot X,
Ferretti L, Fraser C (2023).“Epidemiological Impacts of the NHS COVID-19
App in England and Wales throughout Its First Year.”<em>Nat Commun</em>,
<b>14</b>,
858.<a href="https://doi.org/10.1038/s41467-023-36495-z">doi:10.1038/s41467-023-36495-z</a>.
</p>
</li>
</ol>
<ol start="171">
<li>
<p>
Mughal SR, Niazi SA, Do T, Gilbert SC, Didelot X, Radford DR, Beighton D
(2023).“Genomic Diversity among Actinomyces Naeslundii Strains and
Closely Related Species.”<em>Microorganisms</em>, <b>11</b>,
254.<a href="https://doi.org/10.3390/microorganisms11020254">doi:10.3390/microorganisms11020254</a>.
</p>
</li>
</ol>
<ol start="170">
<li>
<p>
Abdel-Glil MY, Hotzel H, Tomaso H, Didelot X, Brandt C, Seyboldt C,
Linde J, Schwarz S, Neubauer H, El-Adawy H (2023).“Genomic Epidemiology
of Campylobacter Fetus Subsp. Venerealis from Germany.”<em>Front. Vet.
Sci.</em>, <b>9</b>,
1069062.<a href="https://doi.org/10.3389/fvets.2022.1069062">doi:10.3389/fvets.2022.1069062</a>.
</p>
</li>
</ol>
<ol start="169">
<li>
<p>
Dingle KE, Freeman J, Didelot X, Quan TP, Eyre DW, Swann J, Spittal WD,
Clark EV, Jolley KA, Walker AS, Wilcox MH, Crook DW (2023).“Penicillin
Binding Protein Substitutions Cooccur with Fluoroquinolone Resistance in
Epidemic Lineages of Multidrug-Resistant Clostridioides
Difficile.”<em>mBio</em>,
e00243-23.<a href="https://doi.org/10.1128/mbio.00243-23">doi:10.1128/mbio.00243-23</a>.
</p>
</li>
</ol>
<ol start="168">
<li>
<p>
Gomez LRE, Savin C, Andrianaivoarimanana V, Randriantseheno LN, Zhou Z,
Kocher A, Didelot X, Rajerison M, Kühnert D (2023).“Phylogenetic
Analysis of the Origin and Spread of Plague in Madagascar.”<em>PLoS
Neglected Tropical Diseases</em>, <b>17</b>(5),
e0010362.<a href="https://doi.org/10.1371/journal.pntd.0010362">doi:10.1371/journal.pntd.0010362</a>.
</p>
</li>
</ol>
<ol start="167">
<li>
<p>
Medina-Aguayo FJ, Didelot X, Everitt RG (2023).“Speeding up Inference of
Homologous Recombination in Bacteria.”<em>Bayesian Anal.</em>,
<b>-1</b>(-1).<a href="https://doi.org/10.1214/23-BA1388">doi:10.1214/23-BA1388</a>.
</p>
</li>
</ol>
<ol start="166">
<li>
<p>
Chase-Topping M, Dallman TJ, Allison L, Lupolova N, Matthews L, Mitchell
S, Banks CJ, Prentice J, Brown H, Tongue S, Henry M, Evans J, Gunn G,
Hoyle D, McNeilly TN, Fitzgerald S, Smith-Palmer A, Shaaban S, Holmes A,
Hanson M, Woolhouse M, Didelot X, Jenkins C, Gally DL (2023).“Analysis
of Escherichia Coli O157 Strains in Cattle and Humans between Scotland
and England &amp; Wales: Implications for Human Health.”<em>Microbial
Genomics</em>, <b>9</b>,
001090.<a href="https://doi.org/10.1099/mgen.0.001090">doi:10.1099/mgen.0.001090</a>.
</p>
</li>
</ol>
<ol start="165">
<li>
<p>
Torres Ortiz A, Kendall M, Storey N, Hatcher J, Dunn H, Roy S, Williams
R, Williams C, Goldstein RA, Didelot X, Harris K, Breuer J, Grandjean L
(2023).“Within-Host Diversity Improves Phylogenetic and Transmission
Reconstruction of SARS-CoV-2 Outbreaks.”<em>eLife</em>, <b>12</b>,
e84384.<a href="https://doi.org/10.7554/eLife.84384">doi:10.7554/eLife.84384</a>.
</p>
</li>
</ol>
<ol start="164">
<li>
<p>
Yenew B, Ghodousi A, Diriba G, Tesfaye E, Cabibbe AM, Amare M, Moga S,
Alemu A, Dagne B, Sinshaw W, Mollalign H, Meaza A, Tadesse M, Gamtesa
DF, Abebaw Y, Seid G, Zerihun B, Getu M, Chiacchiaretta M, Gaudin C,
Marceau M, Didelot X, Tolera G, Abdella S, Kebede A, Getahun M, Mehammed
Z, Supply P, Cirillo DM (2023).“A Smooth Tubercle Bacillus from Ethiopia
Phylogenetically Close to the Mycobacterium Tuberculosis
Complex.”<em>Nat Commun</em>, <b>14</b>(1),
7519.<a href="https://doi.org/10.1038/s41467-023-42755-9">doi:10.1038/s41467-023-42755-9</a>.
</p>
</li>
</ol>
<ol start="163">
<li>
<p>
Thorell K, Muñoz-Ramírez ZY, Wang D, Sandoval-Motta S, Boscolo Agostini
R, Ghirotto S, Torres RC, HpGP Research Network, Falush D, Camargo MC,
Rabkin CS (2023).“The Helicobacter Pylori Genome Project: Insights into
H. Pylori Population Structure from Analysis of a Worldwide Collection
of Complete Genomes.”<em>Nature Communications</em>, <b>14</b>(1),
8184.<a href="https://doi.org/10.1038/s41467-023-43562-y">doi:10.1038/s41467-023-43562-y</a>.
</p>
</li>
</ol>
<ol start="162">
<li>
<p>
Osnes MN, Brynildsrud OB, Alfsnes K, Van Dorp L, McKeand SA, Ross J,
Town K, DeKorne-Elenbaas J, Van Dam A, Bruisten S, De Blasio BF, Caugant
DA, Grad YH, Williamson DA, Balloux F, Didelot X, Eldholm V (2023).“The
Inter-Continental Population Dynamics of <em>Neisseria</em><em>
Gonorrhoeae</em>.”<em>eLife</em>, <b>12</b>,
RP90062.<a href="https://doi.org/10.7554/eLife.90062.1">doi:10.7554/eLife.90062.1</a>.
</p>
</li>
</ol>
</div>
<div id="section-3" class="section level3">
<h3>2022</h3>
<ol start="161">
<li>
<p>
Larsen J, Raisen CL, Ba X, Sadgrove NJ, Padilla-González GF, Simmonds
MSJ, Loncaric I, Kerschner H, Apfalter P, Hartl R, Deplano A,
Vandendriessche S, ČernáBolfíková B, Hulva P, Arendrup MC, Hare RK,
Barnadas C, Stegger M, Sieber RN, Skov RL, Petersen A, Angen Ø,
Rasmussen SL, Espinosa-Gongora C, Aarestrup FM, Lindholm LJ, Nykäsenoja
SM, Laurent F, Becker K, Walther B, Kehrenberg C, Cuny C, Layer F,
Werner G, Witte W, Stamm I, Moroni P, Jørgensen HJ, deLencastre H,
Cercenado E, García-Garrote F, Börjesson S, Hæggman S, Perreten V, Teale
CJ, Waller AS, Pichon B, Curran MD, Ellington MJ, Welch JJ, Peacock SJ,
Seilly DJ, Morgan FJE, Parkhill J, Hadjirin NF, Lindsay JA, Holden MTG,
Edwards GF, Foster G, Paterson GK, Didelot X, Holmes MA, Harrison EM,
Larsen AR (2022).“Emergence of Methicillin Resistance Predates the
Clinical Use of Antibiotics.”<em>Nature</em>, <b>602</b>,
135–141.<a href="https://doi.org/10.1038/s41586-021-04265-w">doi:10.1038/s41586-021-04265-w</a>.
</p>
</li>
</ol>
<ol start="160">
<li>
<p>
Wang L, Didelot X, Bi Y, Gao GF (2022).“SARS-CoV-2 Transmissibility
Compared between Variants of Concern and Vaccination
Status.”<em>Briefings in Bioinformatics</em>, <b>23</b>(2),
1–12.<a href="https://doi.org/10.1093/bib/bbab594">doi:10.1093/bib/bbab594</a>.
</p>
</li>
</ol>
<ol start="159">
<li>
<p>
Fountain-Jones NM, Kraberger S, Gagne RB, Gilbertson MLJ, Trumbo DR,
Charleston M, Salerno PE, Chris Funk W, Crooks K, Logan K, Alldredge M,
Dellicour S, Baele G, Didelot X, VandeWoude S, Carver S, Craft ME
(2022).“Hunting Alters Viral Transmission and Evolution in a Large
Carnivore.”<em>Nature Ecology &amp; Evolution</em>, <b>6</b>,
174–182.<a href="https://doi.org/10.1038/s41559-021-01635-5">doi:10.1038/s41559-021-01635-5</a>.
</p>
</li>
</ol>
<ol start="158">
<li>
<p>
Ledda A, Cummins M, Shaw LP, Jauneikaite E, Cole K, Lasalle F, Barry D,
Turton J, Rosmarin C, Anaraki S, Wareham D, Stoesser N, Paul J, Manuel
R, Cherian BP, Didelot X (2022).“Hospital Outbreak of
Carbapenem-Resistant Enterobacterales Associated with a blaOXA-48
Plasmid Carried Mostly by Escherichia Coli ST399.”<em>Microbial
Genomics</em>, <b>8</b>(4),
1–11.<a href="https://doi.org/10.1099/mgen.0.000675">doi:10.1099/mgen.0.000675</a>.
</p>
</li>
</ol>
<ol start="157">
<li>
<p>
Whittles LK, Didelot X, White PJ (2022).“Public Health Impact and
Cost-Effectiveness of Gonorrhoea Vaccination: An Integrated
Transmission-Dynamic Health-Economic Modelling Analysis.”<em>Lancet
Infectious Diseases</em>, <b>22</b>,
1030–1041.<a href="https://doi.org/10.1016/s1473-3099%2821%2900744-1">doi:10.1016/s1473-3099(21)00744-1</a>.
</p>
</li>
</ol>
<ol start="156">
<li>
<p>
Carson J, Ledda A, Ferretti L, Keeling M, Didelot X (2022).“The Bounded
Coalescent Model: Conditioning a Genealogy on a Minimum Root
Date.”<em>Journal of Theoretical Biology</em>, <b>548</b>,
111186.<a href="https://doi.org/10.1016/j.jtbi.2022.111186">doi:10.1016/j.jtbi.2022.111186</a>.
</p>
</li>
</ol>
<ol start="155">
<li>
<p>
Volk D, Yang-Turner F, Didelot X, Crook DW, Wyllie D (2022).“Catwalk:
Identifying Closely Related Sequences in Large Microbial Sequence
Databases.”<em>Microbial Genomics</em>, <b>8</b>(6),
1–6.<a href="https://doi.org/10.1099/mgen.0.000850">doi:10.1099/mgen.0.000850</a>.
</p>
</li>
</ol>
<ol start="154">
<li>
<p>
Willgert K, Didelot X, Surendran-Nair M, Kuchipudi SV, Ruden RM, Yon M,
Nissly RH, Vandegrift KJ, Nelli RK, Li L, Jayarao BM, Levine N, Olsen
RJ, Davis JJ, Musser JM, Hudson PJ, Kapur V, Conlan AJK
(2022).“Transmission History of SARS-CoV-2 in Humans and White-Tailed
Deer.”<em>Scientific Reports</em>, <b>12</b>(1),
12094.<a href="https://doi.org/10.1038/s41598-022-16071-z">doi:10.1038/s41598-022-16071-z</a>.
</p>
</li>
</ol>
<ol start="153">
<li>
<p>
Didelot X, Evans CM (2022).“London Parochial Burial Records from 1563 to
1665 Indicate Higher Plague Death Rates for Males than Females: Some
Possible Demographic and Social Explanations.”<em>PLoS One</em>,
<b>17</b>(8),
e0272278.<a href="https://doi.org/10.1371/journal.pone.0272278">doi:10.1371/journal.pone.0272278</a>.
</p>
</li>
</ol>
<ol start="152">
<li>
<p>
Didelot X, Parkhill J (2022).“A Scalable Analytical Approach from
Bacterial Genomes to Epidemiology.”<em>Philosophical Transactions of the
Royal Society B: Biological Sciences</em>, <b>377</b>(1861),
20210246.<a href="https://doi.org/10.1098/rstb.2021.0246">doi:10.1098/rstb.2021.0246</a>.
</p>
</li>
</ol>
<div style="page-break-after: always;"></div>
</div>
<div id="section-4" class="section level3">
<h3>2021</h3>
<ol start="151">
<li>
<p>
Didelot X, Siveroni I, Volz EM (2021).“Additive Uncorrelated Relaxed
Clock Models for the Dating of Genomic Epidemiology
Phylogenies.”<em>Molecular Biology and Evolution</em>, <b>38</b>,
307–317.<a href="https://doi.org/10.1093/molbev/msaa193">doi:10.1093/molbev/msaa193</a>.
</p>
</li>
</ol>
<ol start="150">
<li>
<p>
Didelot X, Kendall M, Xu Y, White PJ, McCarthy N (2021).“Genomic
Epidemiology Analysis of Infectious Disease Outbreaks Using
TransPhylo.”<em>Current Protocols</em>, <b>1</b>,
e60.<a href="https://doi.org/10.1002/cpz1.60">doi:10.1002/cpz1.60</a>.
</p>
</li>
</ol>
<ol start="149">
<li>
<p>
Didelot X (2021).“Phylogenetic Methods for Genome-Wide Association
Studies in Bacteria.”In <em>Bacterial Pangenomics: Methods and
Protocols</em>, volume 2242, 205–220.Springer Nature.ISBN
978-1-0716-1099-2,
<a href="https://doi.org/10.1007/978-1-0716-1099-2_13">doi:10.1007/978-1-0716-1099-2_13</a>.
</p>
</li>
</ol>
<ol start="148">
<li>
<p>
Lassalle F, Dastgheib SMM, Zhao F, Zhang J, Verbarg S, Fruhling A,
Brinkmann H, Osborne TH, Sikorski J, Balloux F, Didelot X, Santini JM,
Petersen J (2021).“Phylogenomic Analysis Reveals the Basis of Adaptation
of Pseudorhizobium Species to Extreme Environments and Supports a
Taxonomic Revision of the Genus.”<em>Systematic and Applied
Microbiology</em>, <b>44</b>,
126165.<a href="https://doi.org/10.1016/j.syapm.2020.126165">doi:10.1016/j.syapm.2020.126165</a>.
</p>
</li>
</ol>
<ol start="147">
<li>
<p>
Wang L, Didelot X, Bi Y, Gao GF (2021).“Assessing the Extent of
Community Spread Caused by Mink-Derived SARS-CoV-2 Variants.”<em>The
Innovation</em>, <b>2</b>(3),
100128.<a href="https://doi.org/10.1016/j.xinn.2021.100128">doi:10.1016/j.xinn.2021.100128</a>.
</p>
</li>
</ol>
<ol start="146">
<li>
<p>
Knight DR, Imwattana K, Kullin B, Guerrero-Araya E, Paredes-Sabja D,
Didelot X, Dingle KE, Eyre DW, Rodríguez C, Riley TV (2021).“Major
Genetic Discontinuity and Novel Toxigenic Species in Clostridioides
Difficile Taxonomy.”<em>eLife</em>, <b>10</b>,
e64325.<a href="https://doi.org/10.7554/eLife.64325">doi:10.7554/eLife.64325</a>.
</p>
</li>
</ol>
<ol start="145">
<li>
<p>
Helekal D, Ledda A, Volz E, Wyllie D, Didelot X (2021).“Bayesian
Inference of Clonal Expansions in a Dated Phylogeny.”<em>Systematic
Biology</em>, <b>71</b>,
1073–1087.<a href="https://doi.org/10.1093/sysbio/syab095">doi:10.1093/sysbio/syab095</a>.
</p>
</li>
</ol>
<ol start="144">
<li>
<p>
Torres Ortiz A, Coronel J, Vidal JR, Bonilla C, Moore DAJ, Gilman RH,
Balloux F, Kon OM, Didelot X, Grandjean L (2021).“Genomic Signatures of
Pre-Resistance in Mycobacterium Tuberculosis.”<em>Nature
Communications</em>, <b>12</b>(1),
7312.<a href="https://doi.org/10.1038/s41467-021-27616-7">doi:10.1038/s41467-021-27616-7</a>.
</p>
</li>
</ol>
<ol start="143">
<li>
<p>
Yang C, Cui Y, Didelot X, Yang R, Falush D (2021).“Why Panmictic
Bacteria Are Rare.”<em>bioRxiv</em>,
385336.<a href="https://doi.org/10.1101/385336">doi:10.1101/385336</a>.
</p>
</li>
</ol>
<ol start="142">
<li>
<p>
Lassalle F, Veber P, Jauneikaite E, Didelot X (2021).“Automated
Reconstruction of All Gene Histories in Large Bacterial Pangenome
Datasets and Search for Co-Evolved Gene Modules with
Pantagruel.”<em>bioRxiv</em>,
586495.<a href="https://doi.org/10.1101/586495">doi:10.1101/586495</a>.
</p>
</li>
</ol>
</div>
<div id="section-5" class="section level3">
<h3>2020</h3>
<ol start="141">
<li>
<p>
Whittles LK, White PJ, Didelot X (2020).“Assessment of the Potential of
Vaccination to Combat Antibiotic Resistance in Gonorrhea: A Modeling
Analysis to Determine Preferred Product Characteristics.”<em>Clinical
Infectious Diseases</em>, <b>71</b>,
1912–1919.<a href="https://doi.org/10.1093/cid/ciz1241">doi:10.1093/cid/ciz1241</a>.
</p>
</li>
</ol>
<ol start="140">
<li>
<p>
Volz EM, Wiuf C, Grad YH, Frost SDW, Dennis AM, Didelot X
(2020).“Identification of Hidden Population Structure in Time-Scaled
Phylogenies.”<em>Systematic Biology</em>, <b>69</b>,
884–896.<a href="https://doi.org/10.1093/sysbio/syaa009">doi:10.1093/sysbio/syaa009</a>.
</p>
</li>
</ol>
<ol start="139">
<li>
<p>
Wang H, Yang C, Sun Z, Zheng W, Zhang W, Yu H, Wu Y, Didelot X, Yang R,
Pan J, Cui Y (2020).“Genomic Epidemiology of Vibrio Cholerae Reveals the
Regional and Global Spread of Two Epidemic Non-Toxigenic
Lineages.”<em>PLoS Neglected Tropical Diseases</em>, <b>14</b>(2),
e0008046.<a href="https://doi.org/10.1371/journal.pntd.0008046">doi:10.1371/journal.pntd.0008046</a>.
</p>
</li>
</ol>
<ol start="138">
<li>
<p>
Lassalle F, Didelot X (2020).“Bacterial Microevolution and the
Pangenome.”In <em>The Pangenome, Diversity, Dynamics and Evolution of
Genomes</em>, 129–149.Springer.ISBN 978-3-030-38281-0,
<a href="https://doi.org/10.1007/978-3-030-38281-0_6">doi:10.1007/978-3-030-38281-0_6</a>.
</p>
</li>
</ol>
<ol start="137">
<li>
<p>
Rodriguez MJ, Moser N, Malpartida CK, Moniri A, Fisarova L, Pennisi I,
Boonyasiri A, Jauneikaite E, Abdolrasouli A, Otter J, Bolt F, Davies F,
Didelot X, Holmes A, Georgiou P (2020).“Rapid Detection of Mobilized
Colistin Resistance Using a Nucleic Acid Based Lab-on-a-Chip Diagnostic
System.”<em>Scientific Reports</em>, <b>10</b>,
8448.<a href="https://doi.org/10.1038/s41598-020-64612-1">doi:10.1038/s41598-020-64612-1</a>.
</p>
</li>
</ol>
<ol start="136">
<li>
<p>
Wang L, Didelot X, Yang J, Wong G, Shi Y, Liu W, Gao GF, Bi Y
(2020).“Inference of Person-to-Person Transmission of COVID-19 Reveals
Hidden Super-Spreading Events during the Early Outbreak
Phase.”<em>Nature Communications</em>, <b>11</b>,
5006.<a href="https://doi.org/10.1038/s41467-020-18836-4">doi:10.1038/s41467-020-18836-4</a>.
</p>
</li>
</ol>
<ol start="135">
<li>
<p>
Janezic S, Dingle K, Alvin J, Accetto T, Didelot X, Crook DW, Lacy DB,
Rupnik M (2020).“Comparative Genomics of Clostridioides Difficile
Toxinotypes Identifies Module-Based Toxin Gene Evolution.”<em>Microbial
Genomics</em>,
10.1099/mgen.0.000449.<a href="https://doi.org/10.1099/mgen.0.000449">doi:10.1099/mgen.0.000449</a>.
</p>
</li>
</ol>
<ol start="134">
<li>
<p>
Vegvari C, Grad YH, White PJ, Didelot X, Whittles LK, Scangarella-oman
NE, Mitrani-gold FS, Dumont E, Perry CR, Gilchrist K, Hossain M,
Mortimer TD, Anderson RM, Gardiner D (2020).“Using Rapid Point-of-Care
Tests to Inform Antibiotic Choice to Mitigate Drug Resistance in
Gonorrhoea.”<em>Euro Surveillance</em>, <b>25</b>,
1900210.<a href="https://doi.org/10.2807/1560-7917.ES.2020.25.43.1900210">doi:10.2807/1560-7917.ES.2020.25.43.1900210</a>.
</p>
</li>
</ol>
<ol start="133">
<li>
<p>
Fountain-Jones NM, Appaw RC, Carver S, Didelot X, Volz E, Charleston M
(2020).“Emerging Phylogenetic Structure of the SARS-CoV-2
Pandemic.”<em>Virus Evolution</em>, <b>6</b>,
veaa082.<a href="https://doi.org/10.1093/ve/veaa082">doi:10.1093/ve/veaa082</a>.
</p>
</li>
</ol>
<ol start="132">
<li>
<p>
Osnes MN, Didelot X, deKorne-Elenbaas J, Alfsnes K, Brynildsrud OB,
Syversen G, Nilsen ØJ, De Blasio BF, Caugant DA, Eldholm V
(2020).“Sudden Emergence of a Neisseria Gonorrhoeae Clade with Reduced
Susceptibility to Extended-Spectrum Cephalosporins,
Norway.”<em>Microbial Genomics</em>,
10.1099/mgen.0.000480.<a href="https://doi.org/10.1099/mgen.0.000480">doi:10.1099/mgen.0.000480</a>.
</p>
</li>
</ol>
<ol start="131">
<li>
<p>
Hennart M, Panunzi L, Rodrigues C, Gaday Q, Baines S, Barros-Pinkelnig
M, Carmi-Leroy A, Dazas M, Wehenkel A, Didelot X, Toubiana J, Badell E,
Brisse S (2020).“Population Genomics and Antimicrobial Resistance in
Corynebacterium Diphtheriae.”<em>Genome Medicine</em>, <b>12</b>,
107.<a href="https://doi.org/10.1101/2020.05.19.101030">doi:10.1101/2020.05.19.101030</a>.
</p>
</li>
</ol>
<div style="page-break-after: always;"></div>
</div>
<div id="section-6" class="section level3">
<h3>2019</h3>
<ol start="130">
<li>
<p>
Whittles LK, White PJ, Didelot X (2019).“A Dynamic Power-Law Sexual
Network Model of Gonorrhoea Outbreaks.”<em>PLoS Computational
Biology</em>, <b>15</b>(3),
e1006748.<a href="https://doi.org/10.1371/journal.pcbi.1006748">doi:10.1371/journal.pcbi.1006748</a>.
</p>
</li>
</ol>
<ol start="129">
<li>
<p>
Didelot X (2019).“Computational Methods in Microbial Population
Genomics.”In Polz MF, Rajora OP (eds.), <em>Population Genomics:
Microorganisms</em>, 3–29.Springer International
Publishing.<a href="https://doi.org/10.1007/13836_2017_3">doi:10.1007/13836_2017_3</a>.
</p>
</li>
</ol>
<ol start="128">
<li>
<p>
Dingle KE, Didelot X, Quan TP, Eyre DW, Stoesser N, Marwick CA, Coia J,
Brown D, Buchanan S, Ijaz UZ, Goswami C, Douce G, Fawley WN, Wilcox MH,
Peto TEA, Walker AS, Crook DW (2019).“A Role for Tetracycline Selection
in Recent Evolution of Agriculture-Associated Clostridium Difficile PCR
Ribotype 078.”<em>mBio</em>, <b>10</b>(2),
e02790-18.<a href="https://doi.org/10.1128/mBio.02790-18">doi:10.1128/mBio.02790-18</a>.
</p>

</li>
</ol>
<ol start="127">
<li>
<p>
vanDorp L, Wang Q, Shaw LP, Acman M, Brynildsrud OB, Eldholm V, Wang R,
Gao H, Yin Y, Chen H, Ding C, Farrer RA, Didelot X, Balloux F, Wang H
(2019).“Rapid Phenotypic Evolution in Multidrug-Resistant Klebsiella
Pneumoniae Hospital Outbreak Strains.”<em>Microbial Genomics</em>,
<b>5</b>,
10.1099/mgen.0.000263.<a href="https://doi.org/10.1099/mgen.0.000263">doi:10.1099/mgen.0.000263</a>.
</p>
</li>
</ol>
<ol start="126">
<li>
<p>
Eyre DW, Didelot X, Buckley AM, Freeman J, Moura IB, Crook DW, Peto TE,
Walker AS, Wilcox MH, Dingle KE (2019).“Clostridium Difficile Trehalose
Metabolism Variants Are Common and Not Associated with Adverse Patient
Outcomes When Variably Present in the Same
Lineage.”<em>EBioMedicine</em>, <b>43</b>,
347–355.<a href="https://doi.org/10.1016/j.ebiom.2019.04.038">doi:10.1016/j.ebiom.2019.04.038</a>.
</p>
</li>
</ol>
<ol start="125">
<li>
<p>
Moniri A, Rodriguez-Manzano J, Malpartida-Cardenas K, Yu L, Didelot X,
Holmes A, Georgiou P (2019).“Framework for DNA Quantification and
Outlier Detection Using Multidimensional Standard Curves.”<em>Analytical
Chemistry</em>, <b>91</b>,
7426–7434.<a href="https://doi.org/10.1021/acs.analchem.9b01466">doi:10.1021/acs.analchem.9b01466</a>.
</p>
</li>
</ol>
<ol start="124">
<li>
<p>
Ailloud F, Didelot X, Woltemate S, Pfaffinger G, Overmann J, Bader RC,
Schulz C, Malfertheiner P, Suerbaum S (2019).“Within-Host Evolution of
Helicobacter Pylori Shaped by Niche-Specific Adaptation, Intragastric
Migrations and Selective Sweeps.”<em>Nature Communications</em>,
<b>10</b>(1),
2273.<a href="https://doi.org/10.1038/s41467-019-10050-1">doi:10.1038/s41467-019-10050-1</a>.
</p>
</li>
</ol>
<ol start="123">
<li>
<p>
Ozer EA, Nnah E, Didelot X, Whitaker RJ, Hauser AR (2019).“The
Population Structure of Pseudomonas Aeruginosa Is Characterized by
Genetic Isolation of exoU+ and exoS+ Lineages.”<em>Genome Biology and
Evolution</em>, <b>11</b>,
1780–1796.<a href="https://doi.org/10.1093/gbe/evz119">doi:10.1093/gbe/evz119</a>.
</p>
</li>
</ol>
<ol start="122">
<li>
<p>
Didelot X, Pouwels KB (2019).“Machine-Learning-Assisted Selection of
Antibiotic Prescription.”<em>Nature Medicine</em>, <b>25</b>,
1033–1034.<a href="https://doi.org/10.1038/s41591-019-0517-0">doi:10.1038/s41591-019-0517-0</a>.
</p>
</li>
</ol>
<ol start="121">
<li>
<p>
Criscuolo A, Issenhuth-Jeanjean S, Didelot X, Thorell K, Hale J,
Parkhill J, Thomson NR, Weill F, Falush D, Brisse S (2019).“The
Speciation and Hybridization History of the Genus
Salmonella.”<em>Microbial Genomics</em>, <b>5</b>,
10.1099/mgen.0.000284.<a href="https://doi.org/10.1099/mgen.0.000284">doi:10.1099/mgen.0.000284</a>.
</p>
</li>
</ol>
<ol start="120">
<li>
<p>
Zhao L, Chen H, Didelot X, Li Z, Li Y, Chen M, Du Y, Zhao H, Li J, Hu Q,
Kan B, Chen M, Pang B (2019).“Co-Existence of Multiple Distinct Lineages
in Vibrio Parahaemolyticus Serotype O4:K12.”<em>Microbial Genomics</em>,
<b>5</b>,
10.1099/mgen.0.000287.<a href="https://doi.org/10.1099/mgen.0.000287">doi:10.1099/mgen.0.000287</a>.
</p>
</li>
</ol>
<ol start="119">
<li>
<p>
Klinkenberg D, Colijn C, Didelot X (2019).“Methods for Outbreaks Using
Genomic Data.”In <em>Handbook of Infectious Disease Data Analysis</em>,
245–263.CRC
Press.<a href="https://doi.org/10.1201/9781315222912">doi:10.1201/9781315222912</a>.
</p>
</li>
</ol>
</div>
<div id="section-7" class="section level3">
<h3>2018</h3>
<ol start="118">
<li>
<p>
Didelot X, Croucher NJ, Bentley SD, Harris SR, Wilson DJ
(2018).“Bayesian Inference of Ancestral Dates on Bacterial Phylogenetic
Trees.”<em>Nucleic Acids Research</em>, <b>46</b>(22),
e134-e134.<a href="https://doi.org/10.1093/nar/gky783">doi:10.1093/nar/gky783</a>.
</p>
</li>
</ol>
<ol start="117">
<li>
<p>
Collins C, Didelot X (2018).“A Phylogenetic Method to Perform
Genome-Wide Association Studies in Microbes That Accounts for Population
Structure and Recombination.”<em>PLoS Computational Biology</em>,
<b>14</b>(2),
e1005958.<a href="https://doi.org/10.1371/journal.pcbi.1005958">doi:10.1371/journal.pcbi.1005958</a>.
</p>
</li>
</ol>
<ol start="116">
<li>
<p>
Volz EM, Didelot X (2018).“Modeling the Growth and Decline of Pathogen
Effective Population Size Provides Insight into Epidemic Dynamics and
Drivers of Antimicrobial Resistance.”<em>Systematic Biology</em>,
<b>67</b>(4),
719–728.<a href="https://doi.org/10.1093/sysbio/syy007">doi:10.1093/sysbio/syy007</a>.
</p>
</li>
</ol>
<ol start="115">
<li>
<p>
Challagundla L, Luo X, Tickler IA, Didelot X, Coleman DC, Shore AC,
Coombs GW, Sordelli DO, Brown EL, Skov R, Larsen R, Reyes J, Robledo IE,
Vazquez GJ, Rivera R, Fey PD, Stevenson K, Wang S, Kreiswirth BN,
Mediavilla JR, Arias CA, Planet PJ, Nolan RL, Tenover FC, Goering RV,
Robinson DA (2018).“Range Expansion and the Origin of USA300 North
American Epidemic Methicillin-Resistant Staphylococcus
Aureus.”<em>mBio</em>, <b>9</b>(1),
e02016-17.<a href="https://doi.org/10.1128/mBio.02016-17">doi:10.1128/mBio.02016-17</a>.
</p>
</li>
</ol>
<ol start="114">
<li>
<p>
Peters J, Cresswell F, Amor L, Cole K, Dean G, Didelot X, De Silva D,
Eyre DW, Paul J (2018).“Whole Genome Sequencing of Neisseria Gonorrhoeae
Reveals Transmission Clusters Involving Patients of Mixed HIV
Serostatus.”<em>Sexually Transmitted Infections</em>, <b>94</b>,
138–143.<a href="https://doi.org/10.1136/sextrans-2017-053198">doi:10.1136/sextrans-2017-053198</a>.
</p>
</li>
</ol>
<ol start="113">
<li>
<p>
Wang R, Van Dorp L, Shaw LP, Bradley P, Wang Q, Wang X, Jin L, Zhang Q,
Liu Y, Rieux A, Dorai-Schneiders T, Weinert LA, Iqbal Z, Didelot X, Wang
H, Balloux F (2018).“The Global Distribution and Spread of the Mobilized
Colistin Resistance Gene Mcr-1.”<em>Nature Communications</em>,
<b>9</b>(1),
1179.<a href="https://doi.org/10.1038/s41467-018-03205-z">doi:10.1038/s41467-018-03205-z</a>.
</p>
</li>
</ol>
<ol start="112">
<li>
<p>
Whittles LK, Didelot X, Grad YH, White PJ (2018).“Testing for Gonorrhoea
Should Routinely Include the Pharynx.”<em>Lancet Infectious
Diseases</em>, <b>18</b>(7),
716–717.<a href="https://doi.org/10.1016/S1473-3099%2818%2930341-4">doi:10.1016/S1473-3099(18)30341-4</a>.
</p>
</li>
</ol>
<ol start="111">
<li>
<p>
Whittles LK, White PJ, Paul J, Didelot X (2018).“Epidemiological Trends
of Antibiotic Resistant Gonorrhea in the United
Kingdom.”<em>Antibiotics</em>, <b>7</b>(3),
60.<a href="https://doi.org/10.3390/antibiotics7030060">doi:10.3390/antibiotics7030060</a>.
</p>
</li>
</ol>
<ol start="110">
<li>
<p>
Séraphin MN, Didelot X, Nolan DJ, May JR, Khan MSR, Murray ER, Salemi M,
Morris JG, Lauzardo M (2018).“Genomic Investigation of a Mycobacterium
Tuberculosis Outbreak Involving Prison and Community Cases in Florida,
United States.”<em>American Journal of Tropical Medicine and
Hygiene</em>, <b>99</b>,
867–874.<a href="https://doi.org/10.4269/ajtmh.17-0700">doi:10.4269/ajtmh.17-0700</a>.
</p>
</li>
</ol>
<ol start="109">
<li>
<p>
Yahara K, Nakayama S, Shimuta K, Lee K, Morita M, Kawahata T, Kuroki T,
Watanabe Y, Ohya H, Yasuda M, Deguchi T, Didelot X, Ohnishi M
(2018).“Genomic Surveillance of Neisseria Gonorrhoeae to Investigate the
Distribution and Evolution of Antimicrobial-Resistance Determinants and
Lineages.”<em>Microbial Genomics</em>, <b>4</b>,
10.1099/mgen.0.000205.<a href="https://doi.org/10.1099/mgen.0.000205">doi:10.1099/mgen.0.000205</a>.
</p>
</li>
</ol>
<ol start="108">
<li>
<p>
Campbell F, Didelot X, Fitzjohn R, Ferguson NM, Cori A, Jombart T
(2018).“Outbreaker2: A Modular Platform for Outbreak
Reconstruction.”<em>BMC Bioinformatics</em>, <b>19</b>,
363.<a href="https://doi.org/10.1186/s12859-018-2330-z">doi:10.1186/s12859-018-2330-z</a>.
</p>
</li>
</ol>
<ol start="107">
<li>
<p>
Pandey A, Cleary DW, Laver JR, Gorringe A, Deasy AM, Dale AP, Morris PD,
Didelot X, Maiden MCJ, Read RC (2018).“Microevolution of Neisseria
Lactamica during Nasopharyngeal Colonisation Induced by Controlled Human
Infection.”<em>Nature Communications</em>, <b>9</b>(1),
4753.<a href="https://doi.org/10.1038/s41467-018-07235-5">doi:10.1038/s41467-018-07235-5</a>.
</p>
</li>
</ol>
<ol start="106">
<li>
<p>
Méric G, Mageiros L, Pensar J, Laabei M, Yahara K, Pascoe B, Kittiwan N,
Tadee P, Post V, Lamble S, Bowden R, Bray JE, Morgenstern M, Jolley KA,
Maiden MCJ, Feil EJ, Didelot X, Miragaia M, deLencastre H, Moriarty TF,
Rohde H, Massey R, Mack D, Corander J, Sheppard SK
(2018).“Disease-Associated Genotypes of the Commensal Skin Bacterium
Staphylococcus Epidermidis.”<em>Nature Communications</em>, <b>9</b>(1),
5034.<a href="https://doi.org/10.1038/s41467-018-07368-7">doi:10.1038/s41467-018-07368-7</a>.
</p>
</li>
</ol>
</div>
<div id="section-8" class="section level3">
<h3>2017</h3>
<ol start="105">
<li>
<p>
Didelot X, Fraser C, Gardy J, Colijn C (2017).“Genomic Infectious
Disease Epidemiology in Partially Sampled and Ongoing
Outbreaks.”<em>Molecular Biology and Evolution</em>, <b>34</b>,
997–1007.<a href="https://doi.org/10.1093/molbev/msw275">doi:10.1093/molbev/msw275</a>.
</p>
</li>
</ol>
<ol start="104">
<li>
<p>
Didelot X, Whittles LK, Hall I (2017).“Model-Based Analysis of an
Outbreak of Bubonic Plague in Cairo in 1801.”<em>Journal of The Royal
Society Interface</em>, <b>14</b>(131),
20170160.<a href="https://doi.org/10.1098/rsif.2017.0160">doi:10.1098/rsif.2017.0160</a>.
</p>
</li>
</ol>
<ol start="103">
<li>
<p>
Ledda A, Price JR, Cole K, Llewelyn MJ, Kearns AM, Crook DW, Paul J,
Didelot X (2017).“Re-Emergence of Methicillin Susceptibility in a
Resistant Lineage of Staphylococcus Aureus.”<em>Journal Of Antimicrobial
Chemotherapy</em>, <b>72</b>,
1285–1288.<a href="https://doi.org/10.1093/jac/dkw570">doi:10.1093/jac/dkw570</a>.
</p>
</li>
</ol>
<ol start="102">
<li>
<p>
Collins C, Didelot X (2017).“Reconstructing the Ancestral Relationships
Between Bacterial Pathogen Genomes.”In <em>Bacterial Pathogenesis:
Methods and Protocols</em>, volume 1535, 109–137.Springer, Clifton,
N.J.<a href="https://doi.org/10.1007/978-1-4939-6673-8_8">doi:10.1007/978-1-4939-6673-8_8</a>,
27914076.
</p>
</li>
</ol>
<ol start="101">
<li>
<p>
Ratmann O, Hodcroft EB, Pickles M, Cori A, Hall M, Lycett S, Colijn C,
Dearlove B, Didelot X, Frost S, Hossain M, Joy JB, Kendall M, Kühnert D,
Leventhal GE, Liang R, Plazzotta G, Poon A, Rasmussen DA, Stadler T,
Volz E, Weis C, Leigh Brown AJ, Fraser C, on behalf of the PANGEA-HIV
consortium (2017).“Phylogenetic Tools for Generalized HIV-1 Epidemics:
Findings from the PANGEA-HIV Methods Comparison.”<em>Molecular Biology
and Evolution</em>, <b>34</b>,
185–203.<a href="https://doi.org/10.1093/molbev/msw217">doi:10.1093/molbev/msw217</a>.
</p>
</li>
</ol>
<ol start="100">
<li>
<p>
Yahara K, Méric G, Taylor AJ, deVries SP, Murray S, Pascoe B, Mageiros
L, Torralbo A, Vidal A, Ridley AM, Komukai S, Wimalarathna H, Cody AJ,
Colles FM, McCarthy ND, Harris D, Bray JE, Jolley KA, Maiden MC, Bentley
SD, Parkhill J, Bayliss CD, Grant AJ, Maskell D, Didelot X, Kelly DJ,
Sheppard SK (2017).“Genome-Wide Association of Functional Traits Linked
with Campylobacter Jejuni Survival from Farm to Fork.”<em>Environmental
Microbiology</em>, <b>19</b>,
361–380.<a href="https://doi.org/10.7287/PEERJ.PREPRINTS.2300V1">doi:10.7287/PEERJ.PREPRINTS.2300V1</a>.
</p>
</li>
</ol>
<ol start="99">
<li>
<p>
Dingle KE, Didelot X, Quan TP, Eyre DW, Stoesser N, Golubchik T, Harding
RM, Wilson DJ, Griffiths D, Vaughan A, Finney JM, Wyllie DH, Oakley SJ,
Fawley WN, Freeman J, Morris K, Martin J, Howard P, Gorbach S, Goldstein
EJC, Citron DM, Hopkins S, Hope R, Johnson AP, Wilcox MH, Peto TEA,
Walker AS, Crook DW, Del Ojo Elias C, Crichton C, Kostiou V, Giess A,
Davies J (2017).“Effects of Control Interventions on Clostridium
Difficile Infection in England: An Observational Study.”<em>Lancet
Infectious Diseases</em>, <b>17</b>(4),
411–421.<a href="https://doi.org/10.1016/S1473-3099%2816%2930514-X">doi:10.1016/S1473-3099(16)30514-X</a>.
</p>
</li>
</ol>
<ol start="98">
<li>
<p>
Yang S, Hemarajata P, Hindler J, Li F, Adisetiyo H, Aldrovandi G, Sebra
R, Kasarskis A, MacCannell D, Didelot X, Russell D, Rubin Z, Humphries R
(2017).“Evolution and Transmission of Carbapenem-Resistant Klebsiella
Pneumoniae Expressing the blaOXA-232 Gene During an Institutional
Outbreak Associated With Endoscopic Retrograde
Cholangiopancreatography.”<em>Clinical Infectious Diseases</em>,
<b>64</b>,
894–901.<a href="https://doi.org/10.1093/cid/ciw876">doi:10.1093/cid/ciw876</a>.
</p>
</li>
</ol>
<ol start="97">
<li>
<p>
Eyre DW, Dingle KE, Didelot X, Quan TP, Peto TEA, Wilcox MH, Walker AS,
Crook DW (2017).“Clostridium Difficile in England: Can We Stop Washing
Our Hands? – Authors’ Reply.”<em>Lancet Infectious Diseases</em>,
<b>17</b>(5),
478–479.<a href="https://doi.org/10.1016/S1473-3099%2817%2930185-8">doi:10.1016/S1473-3099(17)30185-8</a>.
</p>
</li>
</ol>
<ol start="96">
<li>
<p>
Pascoe B, Méric G, Yahara K, Wimalarathna H, Murray S, Hitchings MDM,
Sproston ELE, Carrillo CDC, Taboada EEN, Cooper KKK, Huynh S, Cody AAJ,
Jolley KAK, Maiden MMC, McCarthy NND, Didelot X, Parker CC, Sheppard
SKS, Meric G, Yahara K, Wimaralathna H, Murray S, Hitchings MDM,
Sproston ELE, Carrillo CDC, Taboada EEN, Cooper KKK, Huynh S, Cody AAJ,
Jolley KAK, Maiden MMC, McCarthy NND, Didelot X, Parker CC, Sheppard SKS
(2017).“Local Genes for Local Bacteria: Evidence of Allopatry in the
Genomes of Transatlantic Campylobacter Populations.”<em>Molecular
Ecology</em>, <b>26</b>(17),
4497–4508.<a href="https://doi.org/10.1111/mec.14176">doi:10.1111/mec.14176</a>.
</p>
</li>
</ol>
<ol start="95">
<li>
<p>
Klinkenberg D, Backer JA, Didelot X, Colijn C, Wallinga J
(2017).“Simultaneous Inference of Phylogenetic and Transmission Trees in
Infectious Disease Outbreaks.”<em>PLoS Computational Biology</em>,
<b>13</b>(5),
e1005495.<a href="https://doi.org/10.1371/journal.pcbi.1005495">doi:10.1371/journal.pcbi.1005495</a>.
</p>

</li>
</ol>
<ol start="94">
<li>
<p>
Mostowy RJ, Croucher NJ, De Maio N, Chewapreecha C, Salter SJ, Turner P,
Aanensen DM, Bentley SD, Didelot X, Fraser C (2017).“Pneumococcal
Capsule Synthesis Locus Cps as Evolutionary Hotspot with Potential to
Generate Novel Serotypes by Recombination.”<em>Molecular Biology and
Evolution</em>, <b>34</b>(10),
2537–2554.<a href="https://doi.org/10.1093/molbev/msx173">doi:10.1093/molbev/msx173</a>.
</p>
</li>
</ol>
<ol start="93">
<li>
<p>
Rosner BM, Schielke A, Didelot X, Kops F, Breidenbach J, Willrich N,
Gölz G, Alter T, Stingl K, Josenhans C, Suerbaum S, Stark K (2017).“A
Combined Case-Control and Molecular Source Attribution Study of Human
Campylobacter Infections in Germany, 2011–2014.”<em>Scientific
Reports</em>, <b>7</b>(1),
5139.<a href="https://doi.org/10.1038/s41598-017-05227-x">doi:10.1038/s41598-017-05227-x</a>.
</p>
</li>
</ol>
<ol start="92">
<li>
<p>
Peacock S, Barnes E, Breuer J, Carroll M, Crook D, Didelot X, Fife M,
Fox A, Greatorex J, Goodfellow I, Kellam P, Kind D, Parkhill J, Walker
TM (2017).“Pathogen Genomics.”In <em>Chief Medical Officier Annual
Report</em>, 134–153.Department of Health and Social Care.
</p>
</li>
</ol>
<ol start="91">
<li>
<p>
Pandey AAK, Cleary DWD, Laver JJR, Maiden MCJM, Didelot X, Gorringe A,
Read RCR (2017).“Neisseria Lactamica Y92-1009 Complete Genome
Sequence.”<em>Standards in Genomic Sciences</em>, <b>12</b>(1),
41.<a href="https://doi.org/10.1186/s40793-017-0250-6">doi:10.1186/s40793-017-0250-6</a>.
</p>
</li>
</ol>
<ol start="90">
<li>
<p>
Whittles LK, White PJ, Didelot X (2017).“Estimating the Fitness Benefit
and Cost of Cefixime Resistance in Neisseria Gonorrhoeae to Inform
Prescription Policy: A Modelling Study.”<em>PLoS Medicine</em>,
<b>14</b>(10),
e1002416.<a href="https://doi.org/10.1371/journal.pmed.1002416">doi:10.1371/journal.pmed.1002416</a>.
</p>
</li>
</ol>
</div>
<div id="section-9" class="section level3">
<h3>2016</h3>
<ol start="89">
<li>
<p>
Didelot X, Walker AS, Peto TE, Crook DW, Wilson DJ (2016).“Within-Host
Evolution of Bacterial Pathogens.”<em>Nature Reviews Microbiology</em>,
<b>14</b>,
150–162.<a href="https://doi.org/10.1038/nrmicro.2015.13">doi:10.1038/nrmicro.2015.13</a>.
</p>
</li>
</ol>
<ol start="88">
<li>
<p>
Didelot X, Dordel J, Whittles LK, Collins C, Bilek N, Bishop CJ, White
PJ, Aanensen DM, Parkhill J, Bentley SD, Spratt BG, Harris SR
(2016).“Genomic Analysis and Comparison of Two Gonorrhea
Outbreaks.”<em>mBio</em>, <b>7</b>(3),
e00525-16.<a href="https://doi.org/10.1128/mBio.00525-16">doi:10.1128/mBio.00525-16</a>.
</p>
</li>
</ol>
<ol start="87">
<li>
<p>
Whittles LK, Didelot X (2016).“Epidemiological Analysis of the Eyam
Plague Outbreak of 1665-1666.”<em>Proceedings of the Royal Society
B</em>, <b>283</b>,
20160618.<a href="https://doi.org/10.1098/rspb.2016.0618">doi:10.1098/rspb.2016.0618</a>.
</p>
</li>
</ol>
<ol start="86">
<li>
<p>
Bubendorfer S, Krebes J, Yang I, Hage E, Schulz TF, Bahlawane C, Didelot
X, Suerbaum S (2016).“Genome-Wide Analysis of Chromosomal Import
Patterns after Natural Transformation of Helicobacter Pylori.”<em>Nature
Communications</em>, <b>7</b>,
11995.<a href="https://doi.org/10.1038/ncomms11995">doi:10.1038/ncomms11995</a>.
</p>
</li>
</ol>
<ol start="85">
<li>
<p>
Yahara K, Didelot X, Jolley KA, Kobayashi I, Maiden MCJ, Sheppard SK,
Falush D (2016).“The Landscape of Realized Homologous Recombination in
Pathogenic Bacteria.”<em>Molecular Biology and Evolution</em>,
<b>33</b>,
456–471.<a href="https://doi.org/10.1093/molbev/msv237">doi:10.1093/molbev/msv237</a>.
</p>
</li>
</ol>
<div style="page-break-after: always;"></div>
<ol start="84">
<li>
<p>
Brown T, Didelot X, Wilson DJ, De Maio N (2016).“SimBac: Simulation of
Whole Bacterial Genomes with Homologous Recombination.”<em>Microbial
Genomics</em>, <b>2</b>,
10.1099/mgen.0.000044.<a href="https://doi.org/10.1099/mgen.0.000044">doi:10.1099/mgen.0.000044</a>.
</p>
</li>
</ol>
<ol start="83">
<li>
<p>
Stoesser N, Sheppard A, Pankhurst L, deMaio N, Moore CE, Sebra R, Turner
P, Anson LW, Kasarskis A, Batty EM, Kos V, Wilson DJ, Phetsouvanh R,
Wyllie D, Sokurenko E, Manges AR, Johnson TJ, Price LB, Peto TEA,
Johnson JR, Didelot X, Walker AS, Crook DW (2016).“Evolutionary History
of the Global Emergence of the Escherichia Coli Epidemic Clone
ST131.”<em>mBio</em>, <b>7</b>(2),
e02162-15.<a href="https://doi.org/10.1128/mBio.02162-15.Invited">doi:10.1128/mBio.02162-15.Invited</a>.
</p>
</li>
</ol>
<ol start="82">
<li>
<p>
Wielgoss S, Didelot X, Chaudhuri RR, Liu X, Weedall GD, Velicer GJ, Vos
M (2016).“A Barrier to Homologous Recombination between Sympatric
Strains of the Cooperative Soil Bacterium Myxococcus Xanthus.”<em>ISME
Journal</em>, <b>10</b>,
2468–2477.<a href="https://doi.org/10.1038/ismej.2016.34">doi:10.1038/ismej.2016.34</a>.
</p>
</li>
</ol>
<ol start="81">
<li>
<p>
Hatherell H, Didelot X, Pollock SL, Tang P, Crisan A, Johnston JC,
Colijn C, Gardy J (2016).“Declaring a Tuberculosis Outbreak over with
Genomic Epidemiology.”<em>Microbial Genomics</em>, <b>1</b>,
10.1099/mgen.0.000060.<a href="https://doi.org/10.1099/mgen.0.000060.v1">doi:10.1099/mgen.0.000060.v1</a>.
</p>
</li>
</ol>
<ol start="80">
<li>
<p>
De Silva D, Peters J, Cole K, Cole MJ, Cresswell F, Dean G, Dave J,
Thomas DR, Foster K, Waldram A, Wilson DJ, Didelot X, Grad YH, Crook DW,
Peto TEA, Walker AS, Paul J, Eyre DW (2016).“Whole-Genome Sequencing to
Determine Transmission of Neisseria Gonorrhoeae: An Observational
Study.”<em>Lancet Infectious Diseases</em>, <b>16</b>(11),
1295–1303.<a href="https://doi.org/10.1016/S1473-3099%2816%2930157-8">doi:10.1016/S1473-3099(16)30157-8</a>.
</p>
</li>
</ol>
<ol start="79">
<li>
<p>
Ansari MA, Didelot X (2016).“Bayesian Inference of the Evolution of a
Phenotype Distribution on a Phylogenetic Tree.”<em>Genetics</em>,
<b>204</b>(1),
89–98.<a href="https://doi.org/10.1534/genetics.116.190496">doi:10.1534/genetics.116.190496</a>.
</p>
</li>
</ol>
<ol start="78">
<li>
<p>
Eldholm V, Rieux A, Monteserin J, Lopez JM, Palmero D, Lopez B, Ritacco
V, Didelot X, Balloux F (2016).“Impact of HIV Co-Infection on the
Evolution and Transmission of Multidrug-Resistant
Tuberculosis.”<em>eLife</em>, <b>5</b>,
e16644.<a href="https://doi.org/10.7554/eLife.16644">doi:10.7554/eLife.16644</a>.
</p>
</li>
</ol>
<ol start="77">
<li>
<p>
Joseph SJ, Marti H, Didelot X, Read TD, Dean D (2016).“Tetracycline
Selective Pressure and Homologous Recombination Shape the Evolution of
Chlamydia Suis: A Recently Identified Zoonotic Pathogen.”<em>Genome
Biology and Evolution</em>, <b>8</b>,
2613–2623.<a href="https://doi.org/10.1093/gbe/evw182">doi:10.1093/gbe/evw182</a>.
</p>
</li>
</ol>
<ol start="76">
<li>
<p>
Joseph SJ, Cox D, Wolff B, Morrison SS, Kozak-Muiznieks NA, Frace M,
Didelot X, Castillo-Ramirez S, Winchell J, Read TD, Dean D
(2016).“Dynamics of Genome Change among Legionella
Species.”<em>Scientific Reports</em>, <b>6</b>,
33442.<a href="https://doi.org/10.1038/srep33442">doi:10.1038/srep33442</a>.
</p>
</li>
</ol>
<ol start="75">
<li>
<p>
Didelot X (2016).“Heroic Sacrifice or Tragic Mistake? Revisiting the
Eyam Plague, 350 Years On.”<em>Significance</em>, <b>13</b>(5),
20–25.<a href="https://doi.org/10.1111/j.1740-9713.2016.00961.x">doi:10.1111/j.1740-9713.2016.00961.x</a>.
</p>
</li>
</ol>
</div>
<div id="section-10" class="section level3">
<h3>2015</h3>
<ol start="74">
<li>
<p>
Didelot X, Wilson DJ (2015).“ClonalFrameML: Efficient Inference of
Recombination in Whole Bacterial Genomes.”<em>PLoS Computational
Biology</em>, <b>11</b>(2),
e1004041.<a href="https://doi.org/10.1371/journal.pcbi.1004041">doi:10.1371/journal.pcbi.1004041</a>.
</p>
</li>
</ol>
<ol start="73">
<li>
<p>
Didelot X, Pang B, Zhou Z, McCann A, Ni P, Li D, Achtman M, Kan B
(2015).“The Role of China in the Global Spread of the Current Cholera
Pandemic.”<em>PLoS Genetics</em>, <b>11</b>,
e1005072.<a href="https://doi.org/10.1371/journal.pgen.1005072">doi:10.1371/journal.pgen.1005072</a>.
</p>
</li>
</ol>
<ol start="72">
<li>
<p>
Biek R, Pybus OG, Lloyd-Smith JO, Didelot X (2015).“Measurably Evolving
Pathogens in the Genomic Era.”<em>Trends in Ecology &amp;
Evolution</em>, <b>30</b>,
306–313.<a href="https://doi.org/10.1016/j.tree.2015.03.009">doi:10.1016/j.tree.2015.03.009</a>.
</p>
</li>
</ol>
<ol start="71">
<li>
<p>
Croucher NJ, Didelot X (2015).“The Application of Genomics to Tracing
Bacterial Pathogen Transmission.”<em>Current Opinion in
Microbiology</em>, <b>23</b>,
62–67.<a href="https://doi.org/10.1016/j.mib.2014.11.004">doi:10.1016/j.mib.2014.11.004</a>.
</p>
</li>
</ol>
<ol start="70">
<li>
<p>
Cao Q, Didelot X, Wu Z, Li ZZ, He L, Li Y, Ni M, You Y, Lin X, Li ZZ,
Gong Y, Zheng M, Zhang M, Liu J, Wang W, Bo X, Falush D, Wang S, Zhang J
(2015).“Progressive Genomic Convergence of Two Helicobacter Pylori
Strains during Mixed Infection of a Patient with Chronic
Gastritis.”<em>Gut</em>, <b>64</b>,
554–61.<a href="https://doi.org/10.1136/gutjnl-2014-307345">doi:10.1136/gutjnl-2014-307345</a>,
25007814.
</p>
</li>
</ol>
<ol start="69">
<li>
<p>
Nandi T, Holden M, Didelot X, Mehershahi K, a. Boddey J, Beacham I, Peak
I, Harting J, Baybayan P, Guo Y, Wang S, Lee CH, Sim B, Essex-Lopresti
A, Sarkar-Tyson M, Nelson M, Smither S, Ong C, Aw LT, Chua HH, Michell
S, Studholme DJ, Titball RW, Chen SL, Parkhill J, Tan P
(2015).“Burkholderia Pseudomallei Sequencing Identifies Genomic Clades
with Distinct Recombination, Accessory and Epigenetic
Profiles.”<em>Genome Research</em>, <b>25</b>,
129–141.<a href="https://doi.org/10.1101/gr.177543.114">doi:10.1101/gr.177543.114</a>.
</p>
</li>
</ol>
<ol start="68">
<li>
<p>
Cui Y, Yang X, Didelot X, Guo C, Li D, Yan Y, Zhang Y, Yuan Y, Yang H,
Wang J, Wang J, Song Y, Zhou D, Falush D, Yang R (2015).“Epidemic
Clones, Oceanic Gene Pools and Eco-LD in the Free Living Marine Pathogen
Vibrio Parahaemolyticus.”<em>Molecular Biology and Evolution</em>,
<b>32</b>,
1396–1410.<a href="https://doi.org/10.1093/molbev/msv009">doi:10.1093/molbev/msv009</a>.
</p>
</li>
</ol>
<ol start="67">
<li>
<p>
Ezewudo MN, Joseph S, Castillo-Ramirez S, Dean D, Rio CD, Didelot X,
Selden RF, Shafer WM, Turingan RS, Unemo M, Read TD (2015).“Population
Structure of Neisseria Gonorrhoeae Based on Whole Genome Data and Its
Relationship with Antibiotic Resistance.”<em>PeerJ</em>, <b>3</b>,
e806.<a href="https://doi.org/10.7717/peerj.806">doi:10.7717/peerj.806</a>.
</p>
</li>
</ol>
<ol start="66">
<li>
<p>
Montano V, Didelot X, Foll M, Linz B, Reinhardt R, Suerbaum S, Moodley
Y, Jensen JD (2015).“Worldwide Population Structure, Long Term
Demography, and Local Adaptation of Helicobacter
Pylori.”<em>Genetics</em>, <b>200</b>,
947–963.<a href="https://doi.org/10.1534/genetics.115.176404">doi:10.1534/genetics.115.176404</a>.
</p>
</li>
</ol>
<ol start="65">
<li>
<p>
Achtman M, Zhou Z, Didelot X (2015).“Formal Comment to
Pettengill.”<em>PLoS One</em>, <b>10</b>(8),
e0134435.<a href="https://doi.org/10.1371/journal.pone.0134435">doi:10.1371/journal.pone.0134435</a>.
</p>
</li>
</ol>
<ol start="64">
<li>
<p>
Mathers AJ, Stoesser N, Sheppard AE, Pankhurst L, Giess A, Yeh AJ,
Didelot X, Turner SD, Sebra R, Kasarskis A, Peto T, Crook DW, Sifri CD
(2015).“Klebsiella Pneumoniae Carbapenemase (KPC) Producing K.
Pneumoniae at a Single Institution: Insights into Endemicity from Whole
Genome Sequencing.”<em>Antimicrobial Agents and Chemotherapy</em>,
<b>59</b>,
1656–1663.<a href="https://doi.org/10.1128/AAC.04292-14">doi:10.1128/AAC.04292-14</a>,
25561339.
</p>
</li>
</ol>
<ol start="63">
<li>
<p>
Joseph SJ, Marti H, Didelot X, Castillo-Ramirez S, Read TD, Dean D
(2015).“Chlamydiaceae Genomics Reveals Interspecies Admixture and the
Recent Evolution of Chlamydia Abortus Infecting Lower Mammalian Species
and Humans.”<em>Genome Biology and Evolution</em>, <b>7</b>,
3070–3084.<a href="https://doi.org/10.1093/gbe/evv201">doi:10.1093/gbe/evv201</a>.
</p>
</li>
</ol>
</div>
<div id="section-11" class="section level3">
<h3>2014</h3>
<ol start="62">
<li>
<p>
Didelot X, Gardy J, Colijn C (2014).“Bayesian Inference of Infectious
Disease Transmission from Whole Genome Sequence Data.”<em>Molecular
Biology and Evolution</em>, <b>31</b>,
1869–1879.<a href="https://doi.org/10.1093/molbev/msu121">doi:10.1093/molbev/msu121</a>.
</p>
</li>
</ol>
<ol start="61">
<li>
<p>
Ansari MA, Didelot X (2014).“Inference of the Properties of the
Recombination Process from Whole Bacterial Genomes.”<em>Genetics</em>,
<b>196</b>,
253–265.<a href="https://doi.org/10.1534/genetics.113.157172">doi:10.1534/genetics.113.157172</a>,
24172133.
</p>
</li>
</ol>
<ol start="60">
<li>
<p>
Dingle KE, Elliott B, Robinson E, Griffiths D, Eyre DW, Stoesser N,
Vaughan A, Golubchik T, Fawley WN, Wilcox MH, Peto TEA, Walker AS, Riley
TV, Crook DW, Didelot X (2014).“Evolutionary History of the Clostridium
Difficile Pathogenicity Locus.”<em>Genome Biology and Evolution</em>,
<b>6</b>,
36–52.<a href="https://doi.org/10.1093/gbe/evt204">doi:10.1093/gbe/evt204</a>.
</p>
</li>
</ol>
<ol start="59">
<li>
<p>
Haase JK, Didelot X, Lecuit M, Korkeala H, Achtman M (2014).“The
Ubiquitous Nature of Listeria Monocytogenes Clones: A Large Scale
MultiLocus Sequence Typing Study.”<em>Environmental Microbiology</em>,
<b>16</b>,
405–416.<a href="https://doi.org/10.1111/1462-2920.12342">doi:10.1111/1462-2920.12342</a>,
24274459.
</p>
</li>
</ol>
<ol start="58">
<li>
<p>
Miller R, Price J, Batty E, Didelot X, Wyllie D, Golubchik T, Crook DW,
Paul J, Peto TEA, Wilson DJ, Cule M, Ip C, Day N, Moore C, Bowden R,
Llewelyn M (2014).“Healthcare-Associated Outbreak of
Meticillin-Resistant Staphylococcus Aureus Bacteraemia: Role of a
Cryptic Variant of an Epidemic Clone.”<em>Journal of Hospital
Infection</em>, <b>86</b>,
83–89.<a href="https://doi.org/10.1016/j.jhin.2013.11.007">doi:10.1016/j.jhin.2013.11.007</a>.
</p>
</li>
</ol>
<ol start="57">
<li>
<p>
Krause DJ, Didelot X, Cadillo-Quiroz H, Whitaker RJ
(2014).“Recombination Shapes Genome Architecture in an Organism from the
Archaeal Domain.”<em>Genome Biology and Evolution</em>, <b>6</b>,
170–178.<a href="https://doi.org/10.1093/gbe/evu003">doi:10.1093/gbe/evu003</a>,
24391154.
</p>
</li>
</ol>
<ol start="56">
<li>
<p>
Jombart T, Cori A, Didelot X, Cauchemez S, Fraser C, Ferguson N
(2014).“Bayesian Reconstruction of Disease Outbreaks by Combining
Epidemiologic and Genomic Data.”<em>PLoS Computational Biology</em>,
<b>10</b>,
e1003457.<a href="https://doi.org/10.1371/journal.pcbi.1003457">doi:10.1371/journal.pcbi.1003457</a>.
</p>
</li>
</ol>
<ol start="55">
<li>
<p>
Yahara K, Didelot X, Ansari MA, Sheppard SK, Falush D (2014).“Efficient
Inference of Recombination Hot Regions in Bacterial
Genomes.”<em>Molecular Biology and Evolution</em>, <b>31</b>,
1593–605.<a href="https://doi.org/10.1093/molbev/msu082">doi:10.1093/molbev/msu082</a>,
24586045.
</p>
</li>
</ol>
<ol start="54">
<li>
<p>
O’Brien J, Didelot X, Iqbal Z, Amenga-Etego L, Ahiska B, Falush D
(2014).“A Bayesian Approach to Inferring the Phylogenetic Structure of
Communities from Metagenomic Data.”<em>Genetics</em>, <b>197</b>,
925–937.<a href="https://doi.org/10.1534/genetics.114.161299">doi:10.1534/genetics.114.161299</a>.
</p>
</li>
</ol>
<ol start="53">
<li>
<p>
Jombart T, Aanensen DM, Baguelin M, Birrell P, Cauchemez S, Camacho A,
Colijn C, Collins C, Cori A, Didelot X, Fraser C, Frost S, Hens N,
Hugues J, Höhle M, Opatowski L, Rambaut A, Ratmann O, Soubeyrand S, a.
Suchard M, Wallinga J, Ypma R, Ferguson N (2014).“OutbreakTools: A New
Platform for Disease Outbreak Analysis Using the R
Software.”<em>Epidemics</em>, <b>7</b>,
28–34.<a href="https://doi.org/10.1016/j.epidem.2014.04.003">doi:10.1016/j.epidem.2014.04.003</a>.
</p>
</li>
</ol>
<ol start="52">
<li>
<p>
Everitt RG, Didelot X, Batty EM, Miller RR, Knox K, Young BC, Bowden R,
Auton A, Votintseva A, Larner-Svensson H, Charlesworth J, Golubchik T,
Ip CLC, Godwin H, Fung R, Peto TEA, Walker AS, Crook DW, Wilson DJ
(2014).“Mobile Elements Drive Recombination Hotspots in the Core Genome
of Staphylococcus Aureus.”<em>Nature Communications</em>, <b>5</b>,
3956.<a href="https://doi.org/10.1038/ncomms4956">doi:10.1038/ncomms4956</a>.
</p>
</li>
</ol>
<ol start="51">
<li>
<p>
Krebes J, Didelot X, Kennemann L, Suerbaum S (2014).“Bidirectional
Genomic Exchange between Helicobacter Pylori Strains from a Family in
Coventry, United Kingdom.”<em>International Journal of Medical
Microbiology</em>, <b>304</b>,
1135–1146.<a href="https://doi.org/10.1016/j.ijmm.2014.08.007">doi:10.1016/j.ijmm.2014.08.007</a>.
</p>
</li>
</ol>
<ol start="50">
<li>
<p>
Stoesser N, Giess A, Batty EM, Sheppard AE, Walker AS, Wilson DJ,
Didelot X, Bashir A, Sebra R, Kasarskis A, Sthapit B, Shakya M, Kelly D,
Pollard AJ, Peto TEA, Crook DW, Donnelly P, Thorson S, Amatya P, Joshi S
(2014).“Genome Sequencing of an Extended Series of NDM-Producing
Klebsiella Pneumoniae Neonatal Infections in a Nepali Hospital
Characterizes the Extent of Community Versus Hospital-associated
Transmission in an Endemic Setting.”<em>Antimicrobial Agents and
Chemotherapy</em>, <b>58</b>,
7347–57.<a href="https://doi.org/10.1128/AAC.03900-14">doi:10.1128/AAC.03900-14</a>,
25267672.
</p>
</li>
</ol>
<ol start="49">
<li>
<p>
Elliott B, Dingle KE, Didelot X, Crook DW, Riley TV (2014).“The
Complexity and Diversity of the Pathogenicity Locus in Clostridium
Difficile Clade 5.”<em>Genome Biology and Evolution</em>, <b>6</b>,
3159–3170.<a href="https://doi.org/10.1093/gbe/evu248">doi:10.1093/gbe/evu248</a>.
</p>
</li>
</ol>
</div>
<div id="section-12" class="section level3">
<h3>2013</h3>
<ol start="48">
<li>
<p>
Didelot X, Nell S, Yang I, Woltemate S, vanderMerwe S, Suerbaum S
(2013).“Genomic Evolution and Transmission of Helicobacter Pylori in Two
South African Families.”<em>Proc Natl Acad Sci USA</em>, <b>110</b>(34),
13880–13885.<a href="https://doi.org/10.1073/pnas.1304681110">doi:10.1073/pnas.1304681110</a>.
</p>
</li>
</ol>
<ol start="47">
<li>
<p>
Didelot X (2013).“Genomic Analysis to Improve the Management of
Outbreaks of Bacterial Infection.”<em>Expert Review of Anti-infective
Therapy</em>, <b>11</b>(4),
335–337.<a href="https://doi.org/10.1586/eri.13.15">doi:10.1586/eri.13.15</a>.
</p>
</li>
</ol>
<ol start="46">
<li>
<p>
Sheppard SK, Didelot X, Meric G, Torralbo A, Jolley KA, Kelly DJ,
Bentley SD, Maiden MCJ, Parkhill J, Falush D (2013).“Genome-Wide
Association Study Identifies Vitamin B5 Biosynthesis as a Host
Specificity Factor in Campylobacter.”<em>Proc Natl Acad Sci USA</em>,
<b>110</b>(29),
11923–7.<a href="https://doi.org/10.5061/dryad.28n35."><a
href="doi:10.5061/dryad.28n35"
class="uri">doi:10.5061/dryad.28n35</a>.</a>.
</p>
</li>
</ol>
<ol start="45">
<li>
<p>
Eyre DW, Cule ML, Wilson DJ, Griffiths D, Vaughan A, O’Connor L, Ip CLC,
Golubchik T, Batty EM, Finney JM, Wyllie DH, Didelot X, Piazza P, Bowden
R, Dingle KE, Harding RM, Crook DW, Wilcox MH, Peto TEA, Walker AS
(2013).“Diverse Sources of C. Difficile Infection Identified on
Whole-Genome Sequencing.”<em>New England Journal of Medicine</em>,
<b>369</b>(13),
1195–1205.<a href="https://doi.org/10.1056/NEJMoa1216064">doi:10.1056/NEJMoa1216064</a>,
24066741.
</p>
</li>
</ol>
<ol start="44">
<li>
<p>
Price JR, Didelot X, Crook DW, Llewelyn MJ, Paul J (2013).“Whole Genome
Sequencing in the Prevention and Control of Staphylococcus Aureus
Infection.”<em>Journal of Hospital Infection</em>, <b>83</b>,
14–21.<a href="https://doi.org/10.1016/j.jhin.2012.10.003">doi:10.1016/j.jhin.2012.10.003</a>.
</p>
</li>
</ol>
<ol start="43">
<li>
<p>
Dingle KE, Didelot X, Ansari MA, Eyre DW, Vaughan A, Griffiths D, Ip
CLC, Batty EM, Golubchik T, Bowden R, Jolley KA, Hood DW, Fawley WN,
Walker AS, Peto TEA, Wilcox MH, Crook DW (2013).“Recombinational
Switching of the Clostridium Difficile S-layer and a Novel Glycosylation
Gene Cluster Revealed by Large Scale Whole Genome
Sequencing.”<em>Journal of Infectious Diseases</em>, <b>207</b>,
675–686.<a href="https://doi.org/10.1093/infdis/jis734">doi:10.1093/infdis/jis734</a>.
</p>
</li>
</ol>
<ol start="42">
<li>
<p>
Sheppard SK, Didelot X, Jolley KA, Darling AE, Pascoe B, Meric G, Kelly
DJ, Cody A, Colles FM, Strachan NJC, Ogden ID, Forbes K, French NP,
Carter P, Miller WG, McCarthy ND, Owen R, Litrup E, Egholm M, Affourtit
JP, Bentley SD, Parkhill J, Maiden MCJ, Falush D (2013).“Progressive
Genome-Wide Introgression in Agricultural Campylobacter
Coli.”<em>Molecular Ecology</em>, <b>22</b>,
1051–1064.<a href="https://doi.org/10.1111/mec.12162">doi:10.1111/mec.12162</a>.
</p>
</li>
</ol>
<ol start="41">
<li>
<p>
deMuinck EJ, Lagesen K, Afset JE, Didelot X, Rønningen KS, Rudi K,
Stenseth NC, Trosvik P (2013).“Comparisons of Infant Escherichia Coli
Isolates Link Genomic Profiles with Adaptation to the Ecological
Niche.”<em>BMC Genomics</em>, <b>14</b>,
81.<a href="https://doi.org/10.1186/1471-2164-14-81">doi:10.1186/1471-2164-14-81</a>,
23384204.
</p>
</li>
</ol>
<ol start="40">
<li>
<p>
Read TD, Joseph SJ, Didelot X, Liang B, Patel L, Dean D
(2013).“Comparative Analysis of Chlamydia Psittaci Genomes Reveals the
Recent Emergence of a Pathogenic Lineage with a Broad Host
Range.”<em>mBio</em>, <b>4</b>(2),
e00604-12.<a href="https://doi.org/10.1128/mBio.00604-12">doi:10.1128/mBio.00604-12</a>,
23532978.
</p>
</li>
</ol>
<ol start="39">
<li>
<p>
Golubchik T, Batty EM, Miller RR, Farr H, Young BC, Larner-Svensson H,
Fung R, Godwin H, Knox K, Votintseva A, Everitt RG, Street T, Cule M, Ip
CLC, Didelot X, aPeto TE, Harding RM, Wilson DJ, Crook DW, Bowden R
(2013).“Within-Host Evolution of Staphylococcus Aureus during
Asymptomatic Carriage.”<em>PLoS One</em>, <b>8</b>(5),
e61319.<a href="https://doi.org/10.1371/journal.pone.0061319">doi:10.1371/journal.pone.0061319</a>.
</p>
</li>
</ol>
<ol start="38">
<li>
<p>
Behrens W, Schweinitzer T, Bal J, Dorsch M, Bleich A, Kops F, Brenneke
B, Didelot X, Suerbaum S, Josenhans C (2013).“Role of Energy Sensor TlpD
of Helicobacter Pylori in Gerbil Colonization and Genome Analyses after
Adaptation in the Gerbil.”<em>Infection and Immunity</em>,
<b>81</b>(10),
3534–3551.<a href="https://doi.org/10.1128/IAI.00750-13">doi:10.1128/IAI.00750-13</a>.
</p>
</li>
</ol>
</div>
<div id="section-13" class="section level3">
<h3>2012</h3>
<ol start="37">
<li>
<p>
Didelot X, Méric G, Falush D, Darling AE (2012).“Impact of Homologous
and Non-Homologous Recombination in the Genomic Evolution of Escherichia
Coli.”<em>BMC Genomics</em>, <b>13</b>(1),
256.<a href="https://doi.org/10.1186/1471-2164-13-256">doi:10.1186/1471-2164-13-256</a>.
</p>
</li>
</ol>
<ol start="36">
<li>
<p>
Didelot X, Bowden R, Wilson DJ, Peto TEA, Crook DW (2012).“Transforming
Clinical Microbiology with Bacterial Genome Sequencing.”<em>Nature
Reviews Genetics</em>, <b>13</b>,
601–612.<a href="https://doi.org/10.1038/nrg3226">doi:10.1038/nrg3226</a>.
</p>
</li>
</ol>
<ol start="35">
<li>
<p>
Didelot X, Eyre DW, Cule M, Ip CLC, Ansari MA, Griffiths D, Vaughan A,
O’Connor L, Golubchik T, Batty EM, Piazza P, Wilson DJ, Bowden R,
Donnelly PJ, Dingle KE, Wilcox M, Walker AS, Crook DW, Peto TE, Harding
RM (2012).“Microevolutionary Analysis of Clostridium Difficile Genomes
to Investigate Transmission.”<em>Genome Biology</em>, <b>13</b>,
R118.<a href="https://doi.org/10.1186/gb-2012-13-12-r118">doi:10.1186/gb-2012-13-12-r118</a>.
</p>
</li>
</ol>
<ol start="34">
<li>
<p>
Cadillo-Quiroz H, Didelot X, Held NL, Herrera A, Darling AE, Reno ML,
Krause DJ, Whitaker RJ (2012).“Patterns of Gene Flow Define Species of
Thermophilic Archaea.”<em>PLoS Biology</em>, <b>10</b>(2),
e1001265.<a href="https://doi.org/10.1371/journal.pbio.1001265">doi:10.1371/journal.pbio.1001265</a>.
</p>
</li>
</ol>
<ol start="33">
<li>
<p>
Moccia C, Krebes J, Kulick S, Didelot X, Kraft C, Bahlawane C, Suerbaum
S (2012).“The Nucleotide Excision Repair (NER) System of Helicobacter
Pylori: Role in Mutation Prevention and Chromosomal Import Patterns
after Natural Transformation.”<em>BMC Microbiology</em>, <b>12</b>,
67.<a href="https://doi.org/10.1186/1471-2180-12-67">doi:10.1186/1471-2180-12-67</a>,
22559785.
</p>
</li>
</ol>
<ol start="32">
<li>
<p>
Namouchi A, Didelot X, Scho U, Gicquel B, Rocha EPC (2012).“After the
Bottleneck: Genome-wide Diversification of the Mycobacterium
Tuberculosis Complex by Mutation, Recombination, and Natural
Selection.”<em>Genome Research</em>, <b>22</b>,
721–34.<a href="https://doi.org/10.1101/gr.129544.111.2002">doi:10.1101/gr.129544.111.2002</a>.
</p>
</li>
</ol>
<ol start="31">
<li>
<p>
Young BC, Golubchik T, Batty EM, Fung R, Larner-Svensson H, Votintseva
AA, Miller RR, Godwin H, Knox K, Everitt RG, Iqbal Z, Rimmer AJ, Cule M,
Ip CLC, Didelot X, Harding RM, Donnelly PJ, Peto TEA, Crook DW, Bowden
R, Wilson DJ (2012).“Evolutionary Dynamics of Staphylococcus Aureus
during Progression from Carriage to Disease.”<em>Proc Natl Acad Sci
USA</em>, <b>109</b>,
4550–4555.<a href="https://doi.org/10.1073/pnas.1113219109">doi:10.1073/pnas.1113219109</a>.
</p>
</li>
</ol>
<ol start="30">
<li>
<p>
Zwick ME, Joseph SJ, Didelot X, Chen PE, aBishop-Lilly K, Stewart AC,
Willner K, Lentz S, Nolan N, Thomason MK, Sozhamannan S, Mateczun AJ, Du
L, Read TD (2012).“Genomic Characterization of the Bacillus Cereus Sensu
Lato Species: Backdrop to the Evolution of Bacillus
Anthracis.”<em>Genome Research</em>, <b>22</b>(8),
1512–24.<a href="https://doi.org/10.1101/gr.134437.111">doi:10.1101/gr.134437.111</a>,
22645259.
</p>
</li>
</ol>
<ol start="29">
<li>
<p>
Eyre DW, Golubchik T, Gordon NC, Bowden R, Piazza P, Batty EM, Ip CLC,
Wilson DJ, Didelot X, O’Connor L, Lay R, Buck D, Kearns AM, Shaw A, Paul
J, Wilcox MH, Donnelly PJ, Peto TEA, Walker AS, Crook DW (2012).“A Pilot
Study of Rapid Benchtop Sequencing of Staphylococcus Aureus and
Clostridium Difficile for Outbreak Detection and Surveillance.”<em>BMJ
Open</em>, <b>2</b>,
e001124.<a href="https://doi.org/10.1136/bmjopen-2012-001124">doi:10.1136/bmjopen-2012-001124</a>.
</p>
</li>
</ol>
<ol start="28">
<li>
<p>
Achtman M, Wain J, Weill F, Nair S, Zhou Z, Sangal V, Krauland MG, Hale
J, Harbottle H, Uesbeck A, Dougan G, Harrison LH, Brisse S,
S.entericaMLSTstudygroup T (2012).“Multilocus Sequence Typing as a
Replacement for Serotyping in Salmonella Enterica.”<em>PLoS
Pathogens</em>, <b>8</b>(6),
e1002776.<a href="https://doi.org/10.1371/journal.ppat.1002776">doi:10.1371/journal.ppat.1002776</a>.
</p>
</li>
</ol>
<ol start="27">
<li>
<p>
Joseph SJ, Didelot X, Rothschild J, deVries HJC, Morré SA, Read TD, Dean
D (2012).“Population Genomics of Chlamydia Trachomatis: Insights on
Drift, Selection, Recombination and Population Structure.”<em>Molecular
Biology and Evolution</em>, <b>29</b>,
3933–46.<a href="https://doi.org/10.1093/molbev/mss198">doi:10.1093/molbev/mss198</a>,
22891032.
</p>
</li>
</ol>
<ol start="26">
<li>
<p>
Zhang L, Thomas JC, Didelot X, Robinson DA (2012).“Molecular Signatures
Identify a Candidate Target of Balancing Selection in an arcD-Like Gene
of Staphylococcus Epidermidis.”<em>Journal Molecular Evolution</em>,
<b>75</b>,
43–54.<a href="https://doi.org/10.1007/s00239-012-9520-5">doi:10.1007/s00239-012-9520-5</a>,
23053194.
</p>
</li>
</ol>
</div>
<div id="section-14" class="section level3">
<h3>2011</h3>
<ol start="25">
<li>
<p>
Didelot X, Bowden R, Street T, Golubchik T, Spencer C, McVean G, Sangal
V, Anjum MF, Achtman M, Falush D, Donnelly PJ (2011).“Recombination and
Population Structure in Salmonella Enterica.”<em>PLoS Genetics</em>,
<b>7</b>(7),
e1002191.<a href="https://doi.org/10.1371/journal.pgen.1002191">doi:10.1371/journal.pgen.1002191</a>.
</p>
</li>
</ol>
<ol start="24">
<li>
<p>
Didelot X, Everitt RG, Johansen AM, Lawson DJ (2011).“Likelihood-Free
Estimation of Model Evidence.”<em>Bayesian Analysis</em>, <b>6</b>,
48–76.<a href="https://doi.org/10.1214/11-BA602">doi:10.1214/11-BA602</a>.
</p>
</li>
</ol>
<ol start="23">
<li>
<p>
Dingle KE, Griffiths D, Didelot X, Evans J, Vaughan A, Kachrimanidou M,
Stoesser N, Jolley KA, Golubchik T, Harding RM, Peto TEA, Fawley W,
Walker AS, Wilcox M, Crook DW (2011).“Clinical Clostridium Difficile:
Clonality and Pathogenicity Locus Diversity.”<em>PLoS One</em>,
<b>6</b>(5),
e19993.<a href="https://doi.org/10.1371/journal.pone.0019993">doi:10.1371/journal.pone.0019993</a>.
</p>
</li>
</ol>
<ol start="22">
<li>
<p>
Gripp E, Hlahla D, Didelot X, Kops F, Maurischat S, Tedin K, Alter T,
Ellerbroek L, Schreiber K, Schomburg D, Janssen T, Bartholomäus P,
Hofreuter D, Woltemate S, Uhr M, Brenneke B, Grüning P, Gerlach G,
Wieler L, Suerbaum S, Josenhans C (2011).“Closely Related Campylobacter
Jejuni Strains from Different Sources Reveal a Generalist Rather than a
Specialist Lifestyle.”<em>BMC Genomics</em>, <b>12</b>(1),
584.<a href="https://doi.org/10.1186/1471-2164-12-584">doi:10.1186/1471-2164-12-584</a>,
22122991.
</p>
</li>
</ol>
<ol start="21">
<li>
<p>
Joseph SJ, Didelot X, Gandhi K, Dean D, Read TD (2011).“Interplay of
Recombination and Selection in the Genomes of Chlamydia
Trachomatis.”<em>Biology Direct</em>, <b>6</b>,
28.<a href="https://doi.org/10.1186/1745-6150-6-28">doi:10.1186/1745-6150-6-28</a>.
</p>
</li>
</ol>
<ol start="20">
<li>
<p>
Kennemann L, Didelot X, Aebischer T, Kuhn S, Drescher B, Droege M,
Reinhardt R, Correa P, Meyer TF, Josenhans C, Falush D, Suerbaum S
(2011).“Helicobacter Pylori Genome Evolution during Human
Infection.”<em>Proc Natl Acad Sci USA</em>, <b>108</b>(12),
5033–5038.<a href="https://doi.org/10.1073/pnas.1018444108">doi:10.1073/pnas.1018444108</a>,
21383187.
</p>
</li>
</ol>
<ol start="19">
<li>
<p>
Laukkanen-Ninios R, Didelot X, Jolley KA, Morelli G, Sangal V, Kristo P,
Brehony C, Imori PFM, Fukushima H, Siitonen A, Tseneva G,
Voskressenskaya E, Falcao JP, Korkeala H, Maiden MCJ, Mazzoni CJ,
Carniel E, Skurnik M, Achtman M (2011).“Population Structure of the
Yersinia Pseudotuberculosis Complex According to Multilocus Sequence
Typing.”<em>Environmental Microbiology</em>, <b>13</b>(12),
3114–3127.<a href="https://doi.org/10.1111/j.1462-2920.2011.02588.x">doi:10.1111/j.1462-2920.2011.02588.x</a>.
</p>
</li>
</ol>
</div>
<div id="section-15" class="section level3">
<h3>2010</h3>
<ol start="18">
<li>
<p>
Didelot X, Lawson DJ, Darling AE, Falush D (2010).“Inference of
Homologous Recombination in Bacteria Using Whole-Genome
Sequences.”<em>Genetics</em>, <b>186</b>(4),
1435–49.<a href="https://doi.org/10.1534/genetics.110.120121">doi:10.1534/genetics.110.120121</a>,
20923983.
</p>
</li>
</ol>
<ol start="17">
<li>
<p>
Didelot X (2010).“Sequence-Based Analysis of Bacterial Population
Structures.”In Robinson D, Falush D, Feil EJ (eds.), <em>Bacterial
Population Genetics in Infectious Disease</em>, 37–60.John Wiley
&amp;amp; Sons,
Inc.<a href="https://doi.org/10.1002/9780470600122.ch3">doi:10.1002/9780470600122.ch3</a>.
</p>
</li>
</ol>
<ol start="16">
<li>
<p>
Didelot X, Maiden MCJ (2010).“Impact of Recombination on Bacterial
Evolution.”<em>Trends in Microbiology</em>, <b>18</b>(7),
315–322.<a href="https://doi.org/10.1016/j.tim.2010.04.002">doi:10.1016/j.tim.2010.04.002</a>,
20452218.
</p>
</li>
</ol>
<ol start="15">
<li>
<p>
Morelli G, Didelot X, Kusecek B, Schwarz S, Bahlawane C, Falush D,
Suerbaum S, Achtman M (2010).“Microevolution of Helicobacter Pylori
during Prolonged Infection of Single Hosts and within Families.”<em>PLoS
Genetics</em>, <b>6</b>(7),
e1001036.<a href="https://doi.org/10.1371/journal.pgen.1001036">doi:10.1371/journal.pgen.1001036</a>.
</p>
</li>
</ol>
<ol start="14">
<li>
<p>
Sangal V, Harbottle H, Mazzoni CJ, Helmuth R, Guerra B, Didelot X,
Paglietti B, Rabsch W, Brisse S, Weill F, Roumagnac P, Achtman M
(2010).“Evolution and Population Structure of Salmonella Enterica
Serovar Newport.”<em>Journal of Bacteriology</em>, <b>192</b>(24),
6465–76.<a href="https://doi.org/10.1128/JB.00969-10">doi:10.1128/JB.00969-10</a>,
20935094.
</p>
</li>
</ol>
<div style="page-break-after: always;"></div>
</div>
<div id="section-16" class="section level3">
<h3>2009</h3>
<ol start="13">
<li>
<p>
Didelot X, Darling A, Falush D (2008).“Inferring Genomic Flux in
Bacteria.”<em>Genome Research</em>, <b>19</b>(2),
306–317.<a href="https://doi.org/10.1101/gr.082263.108">doi:10.1101/gr.082263.108</a>.
</p>
</li>
</ol>
<ol start="12">
<li>
<p>
Didelot X, Barker M, Falush D, Priest FG (2009).“Evolution of
Pathogenicity in the Bacillus Cereus Group.”<em>Systematic and Applied
Microbiology</em>, <b>32</b>(2),
81–90.<a href="https://doi.org/10.1016/j.syapm.2009.01.001">doi:10.1016/j.syapm.2009.01.001</a>,
19200684.
</p>
</li>
</ol>
<ol start="11">
<li>
<p>
Didelot X, Lawson DJ, Falush D (2009).“SimMLST: Simulation of
Multi-Locus Sequence Typing Data under a Neutral
Model.”<em>Bioinformatics</em>, <b>25</b>(11),
1442–4.<a href="https://doi.org/10.1093/bioinformatics/btp145">doi:10.1093/bioinformatics/btp145</a>,
19286834.
</p>
</li>
</ol>
<ol start="10">
<li>
<p>
Didelot X, Urwin R, Maiden MCJ, Falush D (2009).“Genealogical Typing of
Neisseria Meningitidis.”<em>Microbiology</em>, <b>155</b>,
3176–86.<a href="https://doi.org/10.1099/mic.0.031534-0">doi:10.1099/mic.0.031534-0</a>.
</p>
</li>
</ol>
<ol start="9">
<li>
<p>
Vos M, Didelot X (2009).“A Comparison of Homologous Recombination Rates
in Bacteria and Archaea.”<em>ISME Journal</em>, <b>3</b>(2),
199–208.<a href="https://doi.org/10.1038/ismej.2008.93">doi:10.1038/ismej.2008.93</a>,
18830278.
</p>
</li>
</ol>
<ol start="8">
<li>
<p>
Zhang J, Zhang F, Didelot X, Bruce KD, Cagampang FR, Vatish M, Hanson M,
Lehnert H, Ceriello A, Byrne CD (2009).“Maternal High Fat Diet during
Pregnancy and Lactation Alters Hepatic Expression of Insulin like Growth
Factor-2 and Key microRNAs in the Adult Offspring.”<em>BMC
Genomics</em>, <b>10</b>,
478.<a href="https://doi.org/10.1186/1471-2164-10-478">doi:10.1186/1471-2164-10-478</a>,
19835573.
</p>
</li>
</ol>
</div>
<div id="section-17" class="section level3">
<h3>2008</h3>
<ol start="7">
<li>
<p>
Didelot X, Falush D (2008).“Bacterial Recombination in Vivo.”In Hensel
M, Schmidt H (eds.), <em>Horizontal Gene Transfer in the Evolution of
Pathogenesis</em>, 23–46.Cambridge University
Press.<a href="https://doi.org/10.1017/CBO9780511541520.003">doi:10.1017/CBO9780511541520.003</a>.
</p>
</li>
</ol>
<ol start="6">
<li>
<p>
Didelot X, Taylor JE, Watkins JC (2008).“A Duality Identity between a
Model of Bacterial Recombination and the Wright-Fisher
Diffusion.”<em>Markov Processes and Related Topics</em>, <b>4</b>,
315–324.<a href="https://doi.org/10.1214/074921708000000453">doi:10.1214/074921708000000453</a>.
</p>
</li>
</ol>
<ol start="5">
<li>
<p>
denBakker HC, Didelot X, Fortes ED, Nightingale KK, Wiedmann M
(2008).“Lineage Specific Recombination Rates and Microevolution in
Listeria Monocytogenes.”<em>BMC Evolutionary Biology</em>, <b>8</b>,
277.<a href="https://doi.org/10.1186/1471-2148-8-277">doi:10.1186/1471-2148-8-277</a>.
</p>
</li>
</ol>
<ol start="4">
<li>
<p>
Kulick S, Moccia C, Didelot X, Falush D, Kraft C, Suerbaum S
(2008).“Mosaic DNA Imports with Interspersions of Recipient Sequence
after Natural Transformation of Helicobacter Pylori.”<em>PLoS One</em>,
<b>3</b>(11),
e3797.<a href="https://doi.org/10.1371/journal.pone.0003797">doi:10.1371/journal.pone.0003797</a>.
</p>
</li>
</ol>
</div>
<div id="section-18" class="section level3">
<h3>2007</h3>
<ol start="3">
<li>
<p>
Didelot X, Achtman M, Parkhill J, Thomson N, Falush D (2007).“A Bimodal
Pattern of Relatedness between the Salmonella Paratyphi A and Typhi
Genomes: Convergence or Divergence by Homologous
Recombination?”<em>Genome Research</em>, <b>17</b>(1),
61–8.<a href="https://doi.org/10.1101/gr.5512906.1">doi:10.1101/gr.5512906.1</a>.
</p>
</li>
</ol>
<ol start="2">
<li>
<p>
Didelot X, Falush D (2007).“Inference of Bacterial Microevolution Using
Multilocus Sequence Data.”<em>Genetics</em>, <b>175</b>(3),
1251–66.<a href="https://doi.org/10.1534/genetics.106.063305">doi:10.1534/genetics.106.063305</a>.
</p>
</li>
</ol>
</div>
<div id="section-19" class="section level3">
<h3>2006</h3>
<ol start="1">
<li>
<p>
Falush D, Torpdahl M, Didelot X, Conrad DF, Wilson DJ, Achtman M
(2006).“Mismatch Induced Speciation in Salmonella: Model and
Data.”<em>Philosophical Transactions of the Royal Society B: Biological
Sciences</em>, <b>361</b>(1475),
2045–53.<a href="https://doi.org/10.1098/rstb.2006.1925">doi:10.1098/rstb.2006.1925</a>.
</p>
</li>
</ol>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
