<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Publications</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />

</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 64px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 69px;
  margin-top: -69px;
}

.section h2 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h3 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h4 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h5 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h6 {
  padding-top: 69px;
  margin-top: -69px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Xavier Didelot</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="research.html">Research</a>
</li>
<li>
  <a href="software.html">Software</a>
</li>
<li>
  <a href="pubs.html">Publications</a>
</li>
<li>
  <a href="bio.html">Background</a>
</li>
<li>
  <a href="contact.html">Contact</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="contact.html">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/xavierdidelot">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/xavierdidelot">
    <span class="fa fa-twitter fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://www.linkedin.com/in/xavier-didelot-538bb531/">
    <span class="fa fa-linkedin fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Publications</h1>

</div>


<h3>
<a name="2018"></a>2018
</h3>
<ul>
<li>
X Didelot. Computational Methods in Microbial Population Genomics In M Polz and O Rajora, editors, <i> Popul. Genomics Microorg.</i>, Springer International Publishing, 2018.<br><a href="http://dx.doi.org/10.1007/13836_2017_3">Download</a> 
</li>
<li>
X Didelot, N Croucher, S Bentley, S Harris, and D Wilson. Bayesian inference of ancestral dates on bacterial phylogenetic trees. <i> bioRxiv</i>, pp. 347385, 2018.<br><a href="http://dx.doi.org/10.1101/347385">Download</a> 
</li>
<li>
L Challagundla, X Luo, I Tickler, X Didelot, D Coleman, A Shore, G Coombs, D Sordelli, E Brown, R Skov, R Larsen, J Reyes, I Robledo, G Vazquez, R Rivera, P Fey, K Stevenson, S Wang, B Kreiswirth, J Mediavilla, C Arias, P Planet, R Nolan, F Tenover, R Goering, and D Robinson. Range Expansion and the Origin of USA300 North American Epidemic Methicillin-Resistant Staphylococcus aureus. <i> MBio</i>, 9(1):e02016–17, 2018.<br><a href="http://mbio.asm.org/content/9/1/e02016-17">Download</a> 
</li>
<li>
C Collins and X Didelot. A phylogenetic method to perform genome-wide association studies in microbes that accounts for population structure and recombination. <i> PLoS Comput. Biol.</i>, 14(2):e1005958, 2018.<br><a href="http://dx.doi.org/10.1101/140798">Download</a> 
</li>
<li>
K Dingle, X Didelot, T Quan, D Eyre, N Stoesser, C Marwick, J Coia, D Brown, S Buchanan, U Ijaz, C Goswami, G Douce, W Fawley, M Wilcox, T Peto, A Walker, and D Crook. A role for tetracycline selection in the evolution of Clostridium difficile PCR-ribotype 078. <i> bioRxiv</i>, pp. 262352, Cold Spring Harbor Laboratory, feb 2018.<br><a href="http://dx.doi.org/10.1101/262352">Download</a> 
</li>
<li>
J Peters, F Cresswell, L Amor, K Cole, G Dean, X Didelot, D De Silva, D Eyre, and J Paul. Whole genome sequencing of Neisseria gonorrhoeae reveals transmission clusters involving patients of mixed HIV serostatus. <i> Sex. Transm. Infect.</i>, 94:138–143, 2018.<br><a href="http://dx.doi.org/10.1136/sextrans-2017-053198">Download</a> 
</li>
<li>
E Volz and X Didelot. Modeling the growth and decline of pathogen effective population size provides insight into epidemic dynamics and drivers of antimicrobial resistance. <i> Syst. Biol.</i>, pp. syy007, 2018.<br><a href="https://doi.org/10.1093/sysbio/syy007">Download</a> 
</li>
<li>
R Wang, L Van Dorp, L Shaw, P Bradley, Q Wang, X Wang, L Jin, Q Zhang, Y Liu, A Rieux, T Dorai-Schneiders, L Weinert, Z Iqbal, X Didelot, H Wang, and F Balloux. The global distribution and spread of the mobilized colistin resistance gene mcr-1. <i> Nat. Commun.</i>, 9(1):1179, 2018.<br><a href="http://dx.doi.org/10.1038/s41467-018-03205-z">Download</a> 
</li>
<li>
L Whittles, P White, and X Didelot. A dynamic power-law sexual network model of gonorrhoea outbreaks. <i> bioRxiv</i>, pp. 1–29, 2018.<br><a href="http://dx.doi.org/10.1101/322875">Download</a> 
</li>
</ul>
<h3>
<a name="2017"></a>2017
</h3>
<ul>
<li>
X Didelot, C Fraser, J Gardy, and C Colijn. Genomic infectious disease epidemiology in partially sampled and ongoing outbreaks. <i> Mol. Biol. Evol.</i>, 34:997–1007, 2017.<br><a href="http://dx.doi.org/10.1093/molbev/msw275">Download</a> 
</li>
<li>
X Didelot, L Whittles, and I Hall. Model-based analysis of an outbreak of bubonic plague in Cairo in 1801. <i> Interface</i>, 14(131):20170160, 2017.<br><a href="http://dx.doi.org/10.1098/rsif.2017.0160">Download</a> 
</li>
<li>
C Collins and X Didelot. Reconstructing the Ancestral Relationships Between Bacterial Pathogen Genomes. <i> Methods Mol. Biol.</i>, 1535:109–137, Clifton, N.J., 2017.<br><a href="http://dx.doi.org/10.1007/978-1-4939-6673-8_8">Download</a> 
</li>
<li>
K Dingle, X Didelot, T Quan, D Eyre, N Stoesser, T Golubchik, R Harding, D Wilson, D Griffiths, A Vaughan, J Finney, D Wyllie, S Oakley, W Fawley, J Freeman, K Morris, J Martin, P Howard, S Gorbach, E Goldstein, D Citron, S Hopkins, R Hope, A Johnson, M Wilcox, T Peto, A Walker, and D Crook. Effects of control interventions on Clostridium difficile infection in England: an observational study. <i> Lancet Infect. Dis.</i>, 17:411–421, Elsevier, 2017.<br><a href="http://dx.doi.org/http://dx.doi.org/10.1016/S1473-3099(16)30514-X">Download</a> 
</li>
<li>
D Eyre, K Dingle, X Didelot, T Quan, T Peto, M Wilcox, A Walker, and D Crook. Clostridium difficile in England: can we stop washing our hands? – Authors’ reply. <i> Lancet Infect. Dis.</i>, 17(5):478–479, Elsevier, may 2017.<br><a href="http://dx.doi.org/10.1016/S1473-3099(17)30185-8">Download</a> 
</li>
<li>
D Klinkenberg, J Backer, X Didelot, C Colijn, and J Wallinga. Simultaneous inference of phylogenetic and transmission trees in infectious disease outbreaks. <i> PLoS Comput. Biol.</i>, 13(5):e1005495, 2017.<br><a href="http://dx.doi.org/10.1371/journal.pcbi.1005495">Download</a> 
</li>
<li>
A Ledda, J Price, K Cole, M Llewelyn, A Kearns, D Crook, J Paul, and X Didelot. Re-emergence of methicillin susceptibility in a resistant lineage of Staphylococcus aureus. <i> J. Antimicrob. Chemother.</i>, 72:1285–1288, Oxford University Press (OUP): Policy B - Oxford Open Option B, 2017.<br><a href="http://dx.doi.org/10.1093/jac/dkw570">Download</a> 
</li>
<li>
R Mostowy, N Croucher, N De Maio, C Chewapreecha, S Salter, P Turner, D Aanensen, S Bentley, X Didelot, and C Fraser. Pneumococcal Capsule Synthesis Locus cps as Evolutionary Hotspot with Potential to Generate Novel Serotypes by Recombination. <i> Mol. Biol. Evol.</i>, 34(10):2537–2554, 2017.<br><a href="http://dx.doi.org/10.1093/molbev/msx173">Download</a> 
</li>
<li>
A Pandey, D Cleary, J Laver, M Maiden, X Didelot, A Gorringe, and R Read. Neisseria lactamica Y92-1009 complete genome sequence. <i> Stand. Genomic Sci.</i>, 12(1):41, Standards in Genomic Sciences, 2017.<br><a href="http://dx.doi.org/10.1186/s40793-017-0250-6">Download</a> 
</li>
<li>
B Pascoe, G Méric, K Yahara, H Wimalarathna, S Murray, M Hitchings, E Sproston, C Carrillo, E Taboada, K Cooper, S Huynh, A Cody, K Jolley, M Maiden, N McCarthy, X Didelot, C Parker, S Sheppard, G Meric, K Yahara, H Wimaralathna, S Murray, M Hitchings, E Sproston, C Carrillo, E Taboada, K Cooper, S Huynh, A Cody, K Jolley, M Maiden, N McCarthy, X Didelot, C Parker, and S Sheppard. Local genes for local bacteria: evidence of allopatry in the genomes of transatlantic Campylobacter populations. <i> Mol. Ecol.</i>, 26(17):4497–4508, 2017.<br><a href="http://dx.doi.org/10.1111/mec.14176">Download</a> 
</li>
<li>
S Peacock, E Barnes, J Breuer, M Carroll, D Crook, X Didelot, M Fife, A Fox, J Greatorex, I Goodfellow, P Kellam, D Kind, J Parkhill, and T Walker. Pathogen genomics In <i> Chief Med. Off. Annu. Rep.</i>, pp. 134–153, 2017.<br><a href="https://www.gov.uk/government/publications/chief-medical-officer-annual-report-2016-generation-genome">Download</a> 
</li>
<li>
O Ratmann, E Hodcroft, M Pickles, A Cori, M Hall, S Lycett, C Colijn, B Dearlove, X Didelot, S Frost, M Hossain, J Joy, M Kendall, D Kühnert, G Leventhal, R Liang, G Plazzotta, A Poon, D Rasmussen, T Stadler, E Volz, C Weis, A Leigh Brown, C Fraser, and on behalf of the PANGEA-HIV consortium. Phylogenetic Tools for Generalized HIV-1 Epidemics: Findings from the PANGEA-HIV Methods Comparison. <i> Mol. Biol. Evol.</i>, 34:185–203, 2017.<br><a href="http://dx.doi.org/10.1093/molbev/msw217">Download</a> 
</li>
<li>
B Rosner, A Schielke, X Didelot, F Kops, J Breidenbach, N Willrich, G Gölz, T Alter, K Stingl, C Josenhans, S Suerbaum, and K Stark. A combined case-control and molecular source attribution study of human Campylobacter infections in Germany, 2011–2014. <i> Sci. Rep.</i>, 7(1):5139, 2017.<br><a href="http://dx.doi.org/10.1038/s41598-017-05227-x">Download</a> 
</li>
<li>
L Whittles, P White, and X Didelot. Quantifying the fitness benefit and cost of cefixime resistance in Neisseria gonorrhoeae to inform prescription policy: A modelling study. <i> PLoS Med.</i>, 14(10):e1002416, 2017.<br><a href="http://dx.doi.org/10.1371/journal.pmed.1002416">Download</a> 
</li>
<li>
K Yahara, G Méric, A Taylor, S de Vries, S Murray, B Pascoe, L Mageiros, A Torralbo, A Vidal, A Ridley, S Komukai, H Wimalarathna, A Cody, F Colles, N McCarthy, D Harris, J Bray, K Jolley, M Maiden, S Bentley, J Parkhill, C Bayliss, A Grant, D Maskell, X Didelot, D Kelly, and S Sheppard. Genome-wide association of functional traits linked with Campylobacter jejuni survival from farm to fork. <i> Environ. Microbiol.</i>, 19:361–380, 2017.<br><a href="http://dx.doi.org/10.7287/PEERJ.PREPRINTS.2300V1">Download</a> 
</li>
<li>
S Yang, P Hemarajata, J Hindler, F Li, H Adisetiyo, G Aldrovandi, R Sebra, A Kasarskis, D MacCannell, X Didelot, D Russell, Z Rubin, and R Humphries. Evolution and Transmission of Carbapenem-Resistant Klebsiella pneumoniae Expressing the blaOXA-232 Gene During an Institutional Outbreak Associated With Endoscopic Retrograde Cholangiopancreatography. <i> Clin. Infect. Dis.</i>, 64:894–901, 2017.<br><a href="http://dx.doi.org/10.1093/cid/ciw876">Download</a> 
</li>
</ul>
<h3>
<a name="2016"></a>2016
</h3>
<ul>
<li>
X Didelot. Heroic sacrifice or tragic mistake? Revisiting the Eyam plague, 350 years on. <i> Significance</i>, 13(5):20–25, 2016.<br><a href="http://dx.doi.org/10.1111/j.1740-9713.2016.00961.x">Download</a> 
</li>
<li>
X Didelot, A Walker, T Peto, D Crook, and D Wilson. Within-host evolution of bacterial pathogens. <i> Nat. Rev. Microbiol.</i>, 14:150–162, 2016.<br><a href="http://dx.doi.org/10.1038/nrmicro.2015.13">Download</a> 
</li>
<li>
X Didelot, J Dordel, L Whittles, C Collins, N Bilek, C Bishop, P White, D Aanensen, J Parkhill, S Bentley, B Spratt, and S Harris. Genomic Analysis and Comparison of Two Gonorrhea Outbreaks. <i> MBio</i>, 7(3):e00525–16, 2016.<br><a href="http://dx.doi.org/10.1128/mBio.00525-16">Download</a> 
</li>
<li>
M Ansari and X Didelot. Bayesian Inference of the Evolution of a Phenotype Distribution on a Phylogenetic Tree. <i> Genetics</i>, 204:89–98, 2016.<br><a href="http://dx.doi.org/10.1101/040980">Download</a> 
</li>
<li>
T Brown, X Didelot, D Wilson, and N De Maio. SimBac: simulation of whole bacterial genomes with homologous recombination. <i> Microb. Genomics</i>, 2:10.1099/mgen.0.000044, 2016.<br><a href="http://dx.doi.org/10.1099/mgen.0.000044">Download</a> 
</li>
<li>
S Bubendorfer, J Krebes, I Yang, E Hage, T Schulz, C Bahlawane, X Didelot, and S Suerbaum. Genome-wide analysis of chromosomal import patterns after natural transformation of Helicobacter pylori. <i> Nat. Commun.</i>, 7:11995, 2016.<br><a href="http://dx.doi.org/10.1038/ncomms11995">Download</a> 
</li>
<li>
V Eldholm, A Rieux, J Monteserin, J Lopez, D Palmero, B Lopez, V Ritacco, X Didelot, and F Balloux. Impact of HIV co-infection on the evolution and transmission of multidrug-resistant tuberculosis. <i> Elife</i>, 5:e16644, 2016.<br><a href="http://dx.doi.org/10.7554/eLife.16644">Download</a> 
</li>
<li>
H Hatherell, X Didelot, S Pollock, P Tang, A Crisan, J Johnston, C Colijn, and J Gardy. Declaring a tuberculosis outbreak over with genomic epidemiology. <i> Microb. Genomics</i>, 1:10.1099/mgen.0.000060, 2016.<br><a href="http://mgen.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000060.v1">Download</a> 
</li>
<li>
S Joseph, D Cox, B Wolff, S Morrison, N Kozak-Muiznieks, M Frace, X Didelot, S Castillo-Ramirez, J Winchell, T Read, and D Dean. Dynamics of genome change among Legionella species. <i> Sci. Rep.</i>, 6:33442, Nature Publishing Group, 2016.<br><a href="http://dx.doi.org/10.1038/srep33442">Download</a> 
</li>
<li>
S Joseph, H Marti, X Didelot, T Read, and D Dean. Tetracycline selective pressure and homologous recombination shape the evolution of Chlamydia suis: a recently identified zoonotic pathogen. <i> Genome Biol. Evol.</i>, 8:2613–2623, 2016.<br><a href="http://dx.doi.org/10.1093/gbe/evw182">Download</a> 
</li>
<li>
N Stoesser, A Sheppard, L Pankhurst, N de Maio, C Moore, R Sebra, P Turner, L Anson, A Kasarskis, E Batty, V Kos, D Wilson, R Phetsouvanh, D Wyllie, E Sokurenko, A Manges, T Johnson, L Price, T Peto, J Johnson, X Didelot, A Walker, and D Crook. Evolutionary history of the global emergence of the Escherichia coli epidemic clone ST131. <i> MBio</i>, 7(2):e02162–15, 2016.<br><a href="http://dx.doi.org/10.1128/mBio.02162-15.Invited">Download</a> 
</li>
<li>
L Whittles and X Didelot. Epidemiological analysis of the Eyam plague outbreak of 1665-1666. <i> Proc. R. Soc. B</i>, 283:20160618, 2016.<br><a href="http://dx.doi.org/10.1098/rspb.2016.0618">Download</a> 
</li>
<li>
S Wielgoss, X Didelot, R Chaudhuri, X Liu, G Weedall, G Velicer, and M Vos. A barrier to homologous recombination between sympatric strains of the cooperative soil bacterium Myxococcus xanthus. <i> ISME J.</i>, 10:2468–2477, 2016.<br><a href="http://dx.doi.org/10.1038/ismej.2016.34">Download</a> 
</li>
<li>
K Yahara, X Didelot, K Jolley, I Kobayashi, M Maiden, S Sheppard, and D Falush. The landscape of realized homologous recombination in pathogenic bacteria. <i> Mol. Biol. Evol.</i>, 33:456–471, 2016.<br><a href="http://dx.doi.org/10.1093/molbev/msv237">Download</a> 
</li>
<li>
D De Silva, J Peters, K Cole, M Cole, F Cresswell, G Dean, J Dave, D Thomas, K Foster, A Waldram, D Wilson, X Didelot, Y Grad, D Crook, T Peto, A Walker, J Paul, and D Eyre. Whole-genome sequencing to determine transmission of Neisseria gonorrhoeae: an observational study. <i> Lancet Infect. Dis.</i>, 16(11):1295–1303, Elsevier Ltd, 2016.<br><a href="http://dx.doi.org/10.1016/S1473-3099(16)30157-8">Download</a> 
</li>
</ul>
<h3>
<a name="2015"></a>2015
</h3>
<ul>
<li>
X Didelot and D Wilson. ClonalFrameML: Efficient Inference of Recombination in Whole Bacterial Genomes. <i> PLOS Comput. Biol.</i>, 11(2):e1004041, feb 2015.<br><a href="http://dx.doi.org/10.1371/journal.pcbi.1004041">Download</a> 
</li>
<li>
X Didelot, B Pang, Z Zhou, A McCann, P Ni, D Li, M Achtman, and B Kan. The Role of China in the Global Spread of the Current Cholera Pandemic. <i> PLOS Genet.</i>, 11:e1005072, 2015.<br><a href="http://dx.doi.org/10.1371/journal.pgen.1005072">Download</a> 
</li>
<li>
M Achtman, Z Zhou, and X Didelot. Formal Comment to Pettengill. <i> PLoS One</i>, 10(8):e0134435, 2015.<br><a href="http://dx.doi.org/10.1371/journal.pone.0134435">Download</a> 
</li>
<li>
R Biek, O Pybus, J Lloyd-Smith, and X Didelot. Measurably evolving pathogens in the genomic era. <i> Trends Ecol. Evol.</i>, 30:306–313, Elsevier Ltd, 2015.<br><a href="http://dx.doi.org/10.1016/j.tree.2015.03.009">Download</a> 
</li>
<li>
Q Cao, X Didelot, Z Wu, Z Li, L He, Y Li, M Ni, Y You, X Lin, Z Li, Y Gong, M Zheng, M Zhang, J Liu, W Wang, X Bo, D Falush, S Wang, and J Zhang. Progressive genomic convergence of two Helicobacter pylori strains during mixed infection of a patient with chronic gastritis. <i> Gut</i>, 64:554–61, 2015.<br><a href="http://dx.doi.org/10.1136/gutjnl-2014-307345">Download</a> 
</li>
<li>
N Croucher and X Didelot. The application of genomics to tracing bacterial pathogen transmission. <i> Curr. Opin. Microbiol.</i>, 23:62–67, 2015.<br><a href="http://dx.doi.org/10.1016/j.mib.2014.11.004">Download</a> 
</li>
<li>
Y Cui, X Yang, X Didelot, C Guo, D Li, Y Yan, Y Zhang, Y Yuan, H Yang, J Wang, J Wang, Y Song, D Zhou, D Falush, and R Yang. Epidemic clones, oceanic gene pools and eco-LD in the free living marine pathogen Vibrio parahaemolyticus. <i> Mol. Biol. Evol.</i>, 32:1396–1410, 2015.<br><a href="http://dx.doi.org/10.1093/molbev/msv009">Download</a> 
</li>
<li>
M Ezewudo, S Joseph, S Castillo-Ramirez, D Dean, C Rio, X Didelot, R Selden, W Shafer, R Turingan, M Unemo, and T Read. Population structure of Neisseria gonorrhoeae based on whole genome data and its relationship with antibiotic resistance. <i> PeerJ</i>, 3:e806, 2015.<br><a href="http://dx.doi.org/10.7717/peerj.806">Download</a> 
</li>
<li>
S Joseph, H Marti, X Didelot, S Castillo-Ramirez, T Read, and D Dean. Chlamydiaceae genomics reveals interspecies admixture and the recent evolution of Chlamydia abortus infecting lower mammalian species and humans. <i> Genome Biol. Evol.</i>, 7:3070–3084, oct 2015.<br><a href="http://dx.doi.org/10.1093/gbe/evv201">Download</a> 
</li>
<li>
A Mathers, N Stoesser, A Sheppard, L Pankhurst, A Giess, A Yeh, X Didelot, S Turner, R Sebra, A Kasarskis, T Peto, D Crook, and C Sifri. Klebsiella pneumoniae carbapenemase (KPC) producing K. pneumoniae at a Single Institution: Insights into Endemicity from Whole Genome Sequencing. <i> Antimicrob. Agents Chemother.</i>, 59:1656–1663, 2015.<br><a href="http://dx.doi.org/10.1128/AAC.04292-14">Download</a> 
</li>
<li>
V Montano, X Didelot, M Foll, B Linz, R Reinhardt, S Suerbaum, Y Moodley, and J Jensen. Worldwide Population Structure, Long Term Demography, and Local Adaptation of Helicobacter pylori. <i> Genetics</i>, 200:947–963, 2015.<br><a href="http://dx.doi.org/10.1534/genetics.115.176404">Download</a> 
</li>
<li>
T Nandi, M Holden, X Didelot, K Mehershahi, J Boddey, I Beacham, I Peak, J Harting, P Baybayan, Y Guo, S Wang, C Lee, B Sim, A Essex-Lopresti, M Sarkar-Tyson, M Nelson, S Smither, C Ong, L Aw, H Chua, S Michell, D Studholme, R Titball, S Chen, J Parkhill, and P Tan. Burkholderia pseudomallei sequencing identifies genomic clades with distinct recombination, accessory and epigenetic profiles. <i> Genome Res.</i>, 25:129–141, 2015.<br><a href="http://dx.doi.org/10.1101/gr.177543.114">Download</a> 
</li>
</ul>
<h3>
<a name="2014"></a>2014
</h3>
<ul>
<li>
X Didelot, J Gardy, and C Colijn. Bayesian inference of infectious disease transmission from whole genome sequence data. <i> Mol. Biol. Evol.</i>, 31:1869–1879, 2014.<br><a href="http://dx.doi.org/10.1093/molbev/msu121">Download</a> 
</li>
<li>
M Ansari and X Didelot. Inference of the Properties of the Recombination Process from Whole Bacterial Genomes. <i> Genetics</i>, 196:253–265, 2014.<br><a href="http://dx.doi.org/10.1534/genetics.113.157172">Download</a> 
</li>
<li>
K Dingle, B Elliott, E Robinson, D Griffiths, D Eyre, N Stoesser, A Vaughan, T Golubchik, W Fawley, M Wilcox, T Peto, A Walker, T Riley, D Crook, and X Didelot. Evolutionary History of the Clostridium difficile Pathogenicity Locus. <i> Genome Biol. Evol.</i>, 6:36–52, 2014.<br><a href="http://dx.doi.org/10.1093/gbe/evt204">Download</a> 
</li>
<li>
B Elliott, K Dingle, X Didelot, D Crook, and T Riley. The complexity and diversity of the Pathogenicity locus in Clostridium difficile clade 5. <i> Genome Biol. Evol.</i>, 6:3159–3170, 2014.<br><a href="http://dx.doi.org/10.1093/gbe/evu248">Download</a> 
</li>
<li>
R Everitt, X Didelot, E Batty, R Miller, K Knox, B Young, R Bowden, A Auton, A Votintseva, H Larner-Svensson, J Charlesworth, T Golubchik, C Ip, H Godwin, R Fung, T Peto, A Walker, D Crook, and D Wilson. Mobile elements drive recombination hotspots in the core genome of Staphylococcus aureus. <i> Nat. Commun.</i>, 5:3956, 2014.<br><a href="http://dx.doi.org/10.1038/ncomms4956">Download</a> 
</li>
<li>
J Haase, X Didelot, M Lecuit, H Korkeala, and M Achtman. The ubiquitous nature of Listeria monocytogenes clones: a large scale MultiLocus Sequence Typing study. <i> Environ. Microbiol.</i>, 16:405–416, 2014.<br><a href="http://dx.doi.org/10.1111/1462-2920.12342">Download</a> 
</li>
<li>
T Jombart, A Cori, X Didelot, S Cauchemez, C Fraser, and N Ferguson. Bayesian Reconstruction of Disease Outbreaks by Combining Epidemiologic and Genomic Data. <i> PLoS Comput. Biol.</i>, 10:e1003457, 2014.<br><a href="http://dx.doi.org/10.1371/journal.pcbi.1003457">Download</a> 
</li>
<li>
T Jombart, D Aanensen, M Baguelin, P Birrell, S Cauchemez, A Camacho, C Colijn, C Collins, A Cori, X Didelot, C Fraser, S Frost, N Hens, J Hugues, M Höhle, L Opatowski, A Rambaut, O Ratmann, S Soubeyrand, M Suchard, J Wallinga, R Ypma, and N Ferguson. OutbreakTools: A new platform for disease outbreak analysis using the R software. <i> Epidemics</i>, 7:28–34, 2014.<br><a href="http://dx.doi.org/10.1016/j.epidem.2014.04.003">Download</a> 
</li>
<li>
D Krause, X Didelot, H Cadillo-Quiroz, and R Whitaker. Recombination shapes genome architecture in an organism from the Archaeal domain. <i> Genome Biol. Evol.</i>, 6:170–178, 2014.<br><a href="http://dx.doi.org/10.1093/gbe/evu003">Download</a> 
</li>
<li>
J Krebes, X Didelot, L Kennemann, and S Suerbaum. Bidirectional genomic exchange between Helicobacter pylori strains from a family in Coventry, United Kingdom. <i> Int. J. Med. Microbiol.</i>, 304:1135–1146, 2014.<br><a href="http://dx.doi.org/10.1016/j.ijmm.2014.08.007">Download</a> 
</li>
<li>
R Miller, J Price, E Batty, X Didelot, D Wyllie, T Golubchik, D Crook, J Paul, T Peto, D Wilson, M Cule, C Ip, N Day, C Moore, R Bowden, and M Llewelyn. Healthcare-associated outbreak of meticillin-resistant Staphylococcus aureus bacteraemia: role of a cryptic variant of an epidemic clone. <i> J. Hosp. Infect.</i>, 86:83–89, 2014.<br><a href="http://dx.doi.org/10.1016/j.jhin.2013.11.007">Download</a> 
</li>
<li>
J O’Brien, X Didelot, Z Iqbal, L Amenga-Etego, B Ahiska, and D Falush. A Bayesian approach to inferring the phylogenetic structure of communities from metagenomic data. <i> Genetics</i>, 197:925–937, 2014.<br><a href="http://www.genetics.org/content/early/2014/05/01/genetics.114.161299.abstract">Download</a> 
</li>
<li>
N Stoesser, A Giess, E Batty, A Sheppard, A Walker, D Wilson, X Didelot, A Bashir, R Sebra, A Kasarskis, B Sthapit, M Shakya, D Kelly, a Pollard, T Peto, D Crook, P Donnelly, S Thorson, P Amatya, and S Joshi. Genome Sequencing of an Extended Series of NDM-Producing Klebsiella pneumoniae Neonatal Infections in a Nepali Hospital Characterizes the Extent of Community Versus Hospital-associated Transmission in an Endemic Setting. <i> Antimicrob. Agents Chemother.</i>, 58:7347–57, 2014.<br><a href="http://dx.doi.org/10.1128/AAC.03900-14">Download</a> 
</li>
<li>
K Yahara, X Didelot, M Ansari, S Sheppard, and D Falush. Efficient inference of recombination hot regions in bacterial genomes. <i> Mol. Biol. Evol.</i>, 31:1593–605, 2014.<br><a href="http://dx.doi.org/10.1093/molbev/msu082">Download</a> 
</li>
</ul>
<h3>
<a name="2013"></a>2013
</h3>
<ul>
<li>
X Didelot. Genomic analysis to improve the management of outbreaks of bacterial infection. <i> Expert Rev. Anti. Infect. Ther.</i>, 11(4):335–337, 2013.<br><a href="http://dx.doi.org/10.1586/eri.13.15">Download</a> 
</li>
<li>
X Didelot, S Nell, I Yang, S Woltemate, S van der Merwe, and S Suerbaum. Genomic evolution and transmission of Helicobacter pylori in two South African families. <i> Proc Natl Acad Sci USA</i>, 110(34):13880–13885, 2013.<br><a href="http://dx.doi.org/10.1073/pnas.1304681110">Download</a> 
</li>
<li>
W Behrens, T Schweinitzer, J Bal, M Dorsch, A Bleich, F Kops, B Brenneke, X Didelot, S Suerbaum, and C Josenhans. Role of energy sensor TlpD of Helicobacter pylori in gerbil colonization and genome analyses after adaptation in the gerbil. <i> Infect. Immun.</i>, 81(10):3534–3551, 2013.<br><a href="http://dx.doi.org/10.1128/IAI.00750-13">Download</a> 
</li>
<li>
K Dingle, X Didelot, M Ansari, D Eyre, A Vaughan, D Griffiths, C Ip, E Batty, T Golubchik, R Bowden, K Jolley, D Hood, W Fawley, A Walker, T Peto, M Wilcox, and D Crook. Recombinational switching of the Clostridium difficile S-layer and a novel glycosylation gene cluster revealed by large scale whole genome sequencing. <i> J. Infect. Dis.</i>, 207:675–686, 2013.<br><a href="http://dx.doi.org/10.1093/infdis/jis734">Download</a> 
</li>
<li>
D Eyre, M Cule, D Wilson, D Griffiths, A Vaughan, L O’Connor, C Ip, T Golubchik, E Batty, J Finney, D Wyllie, X Didelot, P Piazza, R Bowden, K Dingle, R Harding, D Crook, M Wilcox, T Peto, and A Walker. Diverse Sources of C. difficile Infection Identified on Whole-Genome Sequencing. <i> N. Engl. J. Med.</i>, 369(13):1195–1205, 2013.<br><a href="http://dx.doi.org/10.1056/NEJMoa1216064">Download</a> 
</li>
<li>
T Golubchik, E Batty, R Miller, H Farr, B Young, H Larner-Svensson, R Fung, H Godwin, K Knox, A Votintseva, R Everitt, T Street, M Cule, C Ip, X Didelot, T Peto, R Harding, D Wilson, D Crook, and R Bowden. Within-Host Evolution of Staphylococcus aureus during Asymptomatic Carriage. <i> PLoS One</i>, 8(5):e61319, 2013.<br><a href="http://dx.doi.org/10.1371/journal.pone.0061319">Download</a> 
</li>
<li>
J Price, X Didelot, D Crook, M Llewelyn, and J Paul. Whole genome sequencing in the prevention and control of Staphylococcus aureus infection. <i> J. Hosp. Infect.</i>, 83:14–21, 2013.<br><a href="http://dx.doi.org/10.1016/j.jhin.2012.10.003">Download</a> 
</li>
<li>
T Read, S Joseph, X Didelot, B Liang, L Patel, and D Dean. Comparative Analysis of Chlamydia psittaci Genomes Reveals the Recent Emergence of a Pathogenic Lineage with a Broad Host Range. <i> MBio</i>, 4(2):e00604–12, 2013.<br><a href="http://dx.doi.org/10.1128/mBio.00604-12">Download</a> 
</li>
<li>
S Sheppard, X Didelot, G Meric, A Torralbo, K Jolley, D Kelly, S Bentley, M Maiden, J Parkhill, and D Falush. Genome-wide association study identifies vitamin B5 biosynthesis as a host specificity factor in Campylobacter. <i> Proc Natl Acad Sci USA</i>, 110(29):11923–7, 2013.<br><a href="http://dx.doi.org/10.5061/dryad.28n35.">Download</a> 
</li>
<li>
S Sheppard, X Didelot, K Jolley, A Darling, B Pascoe, G Meric, D Kelly, A Cody, F Colles, N Strachan, I Ogden, K Forbes, N French, P Carter, W Miller, N McCarthy, R Owen, E Litrup, M Egholm, J Affourtit, S Bentley, J Parkhill, M Maiden, and D Falush. Progressive genome-wide introgression in agricultural Campylobacter coli. <i> Mol. Ecol.</i>, 22:1051–1064, 2013.<br><a href="http://dx.doi.org/10.1111/mec.12162">Download</a> 
</li>
<li>
E de Muinck, K Lagesen, J Afset, X Didelot, K Rønningen, K Rudi, N Stenseth, and P Trosvik. Comparisons of infant Escherichia coli isolates link genomic profiles with adaptation to the ecological niche. <i> BMC Genomics</i>, 14:81, 2013.<br><a href="http://dx.doi.org/10.1186/1471-2164-14-81">Download</a> 
</li>
</ul>
<h3>
<a name="2012"></a>2012
</h3>
<ul>
<li>
X Didelot, G Méric, D Falush, and A Darling. Impact of homologous and non-homologous recombination in the genomic evolution of Escherichia coli. <i> BMC Genomics</i>, 13(1):256, 2012.<br><a href="http://dx.doi.org/10.1186/1471-2164-13-256">Download</a> 
</li>
<li>
X Didelot, D Eyre, M Cule, C Ip, M Ansari, D Griffiths, A Vaughan, L O’Connor, T Golubchik, E Batty, P Piazza, D Wilson, R Bowden, P Donnelly, K Dingle, M Wilcox, A Walker, D Crook, T Peto, and R Harding. Microevolutionary analysis of Clostridium difficile genomes to investigate transmission. <i> Genome Biol.</i>, 13:R118, 2012.<br><a href="http://dx.doi.org/10.1186/gb-2012-13-12-r118">Download</a> 
</li>
<li>
X Didelot, R Bowden, D Wilson, T Peto, and D Crook. Transforming clinical microbiology with bacterial genome sequencing. <i> Nat. Rev. Genet.</i>, 13:601–612, 2012.<br><a href="http://dx.doi.org/10.1038/nrg3226">Download</a> 
</li>
<li>
M Achtman, J Wain, F Weill, S Nair, Z Zhou, V Sangal, M Krauland, J Hale, H Harbottle, A Uesbeck, G Dougan, L Harrison, S Brisse, and T S. enterica MLST study group. Multilocus Sequence Typing as a Replacement for Serotyping in Salmonella enterica. <i> PLoS Pathog.</i>, 8(6):e1002776, 2012.<br><a href="http://dx.doi.org/10.1371/journal.ppat.1002776">Download</a> 
</li>
<li>
H Cadillo-Quiroz, X Didelot, N Held, A Herrera, A Darling, M Reno, D Krause, and R Whitaker. Patterns of Gene Flow Define Species of Thermophilic Archaea. <i> PLoS Biol.</i>, 10(2):e1001265, 2012.<br><a href="http://dx.doi.org/10.1371/journal.pbio.1001265">Download</a> 
</li>
<li>
D Eyre, T Golubchik, N Gordon, R Bowden, P Piazza, E Batty, C Ip, D Wilson, X Didelot, L O’Connor, R Lay, D Buck, A Kearns, A Shaw, J Paul, M Wilcox, P Donnelly, T Peto, A Walker, and D Crook. A pilot study of rapid benchtop sequencing of Staphylococcus aureus and Clostridium difficile for outbreak detection and surveillance. <i> BMJ Open</i>, 2:e001124, 2012.<br><a href="http://dx.doi.org/10.1136/bmjopen-2012-001124">Download</a> 
</li>
<li>
S Joseph, X Didelot, J Rothschild, H de Vries, S Morré, T Read, and D Dean. Population Genomics of Chlamydia trachomatis: Insights on Drift, Selection, Recombination and Population Structure. <i> Mol. Biol. Evol.</i>, 29:3933–46, 2012.<br><a href="http://dx.doi.org/10.1093/molbev/mss198">Download</a> 
</li>
<li>
C Moccia, J Krebes, S Kulick, X Didelot, C Kraft, C Bahlawane, and S Suerbaum. The nucleotide excision repair (NER) system of Helicobacter pylori: Role in mutation prevention and chromosomal import patterns after natural transformation. <i> BMC Microbiol.</i>, 12:67, 2012.<br><a href="http://dx.doi.org/10.1186/1471-2180-12-67">Download</a> 
</li>
<li>
A Namouchi, X Didelot, U Scho, B Gicquel, and E Rocha. After the bottleneck: Genome-wide diversification of the Mycobacterium tuberculosis complex by mutation, recombination, and natural selection. <i> Genome Res.</i>, 22:721–34, 2012.<br><a href="http://dx.doi.org/10.1101/gr.129544.111.2002">Download</a> 
</li>
<li>
B Young, T Golubchik, E Batty, R Fung, H Larner-svensson, A Votintseva, R Miller, H Godwin, K Knox, R Everitt, Z Iqbal, A Rimmer, M Cule, C Ip, X Didelot, R Harding, P Donnelly, T Peto, D Crook, R Bowden, and D Wilson. Evolutionary dynamics of Staphylococcus aureus during progression from carriage to disease. <i> Proc Natl Acad Sci USA</i>, 109:4550–4555, 2012.<br><a href="http://dx.doi.org/10.1073/pnas.1113219109">Download</a> 
</li>
<li>
L Zhang, J Thomas, X Didelot, and D Robinson. Molecular Signatures Identify a Candidate Target of Balancing Selection in an arcD-Like Gene of Staphylococcus epidermidis. <i> J. Mol. Evol.</i>, 75:43–54, 2012.<br><a href="http://dx.doi.org/10.1007/s00239-012-9520-5">Download</a> 
</li>
<li>
M Zwick, S Joseph, X Didelot, P Chen, K Bishop-Lilly, A Stewart, K Willner, S Lentz, N Nolan, M Thomason, S Sozhamannan, A Mateczun, L Du, and T Read. Genomic characterization of the Bacillus cereus sensu lato species: backdrop to the evolution of Bacillus anthracis. <i> Genome Res.</i>, 22(8):1512–24, 2012.<br><a href="http://dx.doi.org/10.1101/gr.134437.111">Download</a> 
</li>
</ul>
<h3>
<a name="2011"></a>2011
</h3>
<ul>
<li>
X Didelot, R Everitt, A Johansen, and D Lawson. Likelihood-free estimation of model evidence. <i> Bayesian Anal.</i>, 6:48–76, 2011.<br><a href="https://projecteuclid.org/euclid.ba/1339611941">Download</a> 
</li>
<li>
X Didelot, R Bowden, T Street, T Golubchik, C Spencer, G McVean, V Sangal, M Anjum, M Achtman, D Falush, and P Donnelly. Recombination and Population Structure in Salmonella enterica. <i> PLOS Genet.</i>, 7(7):e1002191, 2011.<br><a href="http://dx.doi.org/10.1371/journal.pgen.1002191">Download</a> 
</li>
<li>
K Dingle, D Griffiths, X Didelot, J Evans, A Vaughan, M Kachrimanidou, N Stoesser, K Jolley, T Golubchik, R Harding, T Peto, W Fawley, A Walker, M Wilcox, and D Crook. Clinical Clostridium difficile: Clonality and Pathogenicity Locus diversity. <i> PLoS One</i>, 6(5):e19993, 2011.<br><a href="http://dx.doi.org/10.1371/journal.pone.0019993">Download</a> 
</li>
<li>
E Gripp, D Hlahla, X Didelot, F Kops, S Maurischat, K Tedin, T Alter, L Ellerbroek, K Schreiber, D Schomburg, T Janssen, P Bartholomäus, D Hofreuter, S Woltemate, M Uhr, B Brenneke, P Grüning, G Gerlach, L Wieler, S Suerbaum, and C Josenhans. Closely related Campylobacter jejuni strains from different sources reveal a generalist rather than a specialist lifestyle. <i> BMC Genomics</i>, 12(1):584, 2011.<br><a href="http://dx.doi.org/10.1186/1471-2164-12-584">Download</a> 
</li>
<li>
S Joseph, X Didelot, K Gandhi, D Dean, and T Read. Interplay of recombination and selection in the genomes of Chlamydia trachomatis. <i> Biol. Direct</i>, 6:28, 2011.<br><a href="http://dx.doi.org/10.1186/1745-6150-6-28">Download</a> 
</li>
<li>
L Kennemann, X Didelot, T Aebischer, S Kuhn, B Drescher, M Droege, R Reinhardt, P Correa, T Meyer, C Josenhans, D Falush, and S Suerbaum. Helicobacter pylori genome evolution during human infection. <i> Proc Natl Acad Sci USA</i>, 108(12):5033–5038, 2011.<br><a href="http://dx.doi.org/10.1073/pnas.1018444108">Download</a> 
</li>
<li>
R Laukkanen-Ninios, X Didelot, K Jolley, G Morelli, V Sangal, P Kristo, C Brehony, P Imori, H Fukushima, A Siitonen, G Tseneva, E Voskressenskaya, J Falcao, H Korkeala, M Maiden, C Mazzoni, E Carniel, M Skurnik, and M Achtman. Population structure of the Yersinia pseudotuberculosis complex according to multilocus sequence typing. <i> Environ. Microbiol.</i>, 13(12):3114–3127, 2011.<br><a href="http://dx.doi.org/10.1111/j.1462-2920.2011.02588.x">Download</a> 
</li>
</ul>
<h3>
<a name="2010"></a>2010
</h3>
<ul>
<li>
X Didelot and M Maiden. Impact of recombination on bacterial evolution. <i> Trends Microbiol.</i>, 18(7):315–322, 2010.<br><a href="http://dx.doi.org/10.1016/j.tim.2010.04.002">Download</a> 
</li>
<li>
X Didelot. Sequence-based analysis of bacterial population structures In D Robinson, D Falush, and E Feil, editors, <i> Bact. Popul. Genet. Infect. Dis.</i>, pp. 37–60, John Wiley &amp; Sons, Inc., 2010.<br><a href="http://eu.wiley.com/WileyCDA/WileyTitle/productCd-0470424745,subjectCd-MDD2.html">Download</a> 
</li>
<li>
X Didelot, D Lawson, A Darling, and D Falush. Inference of homologous recombination in bacteria using whole-genome sequences. <i> Genetics</i>, 186(4):1435–49, 2010.<br><a href="http://dx.doi.org/10.1534/genetics.110.120121">Download</a> 
</li>
<li>
G Morelli, X Didelot, B Kusecek, S Schwarz, C Bahlawane, D Falush, S Suerbaum, and M Achtman. Microevolution of Helicobacter pylori during prolonged infection of single hosts and within families. <i> PLOS Genet.</i>, 6(7):e1001036, 2010.<br><a href="http://dx.doi.org/10.1371/journal.pgen.1001036">Download</a> 
</li>
<li>
V Sangal, H Harbottle, C Mazzoni, R Helmuth, B Guerra, X Didelot, B Paglietti, W Rabsch, S Brisse, F Weill, P Roumagnac, and M Achtman. Evolution and Population Structure of Salmonella enterica Serovar Newport. <i> J. Bacteriol.</i>, 192(24):6465–76, 2010.<br><a href="http://dx.doi.org/10.1128/JB.00969-10">Download</a> 
</li>
</ul>
<h3>
<a name="2009"></a>2009
</h3>
<ul>
<li>
X Didelot, A Darling, and D Falush. Inferring genomic flux in bacteria. <i> Genome Res.</i>, 19(2):306–317, 2009.<br><a href="http://dx.doi.org/10.1101/gr.082263.108.clearly">Download</a> 
</li>
<li>
X Didelot, D Lawson, and D Falush. SimMLST: simulation of multi-locus sequence typing data under a neutral model. <i> Bioinformatics</i>, 25(11):1442–4, 2009.<br><a href="http://dx.doi.org/10.1093/bioinformatics/btp145">Download</a> 
</li>
<li>
X Didelot, R Urwin, M Maiden, and D Falush. Genealogical typing of Neisseria meningitidis. <i> Microbiology</i>, 155:3176–86, 2009.<br><a href="http://dx.doi.org/10.1099/mic.0.031534-0">Download</a> 
</li>
<li>
X Didelot, M Barker, D Falush, and F Priest. Evolution of pathogenicity in the Bacillus cereus group. <i> Syst. Appl. Microbiol.</i>, 32(2):81–90, 2009.<br><a href="http://dx.doi.org/10.1016/j.syapm.2009.01.001">Download</a> 
</li>
<li>
M Vos and X Didelot. A comparison of homologous recombination rates in bacteria and archaea. <i> ISME J.</i>, 3(2):199–208, 2009.<br><a href="http://dx.doi.org/10.1038/ismej.2008.93">Download</a> 
</li>
<li>
J Zhang, F Zhang, X Didelot, K Bruce, F Cagampang, M Vatish, M Hanson, H Lehnert, A Ceriello, and C Byrne. Maternal high fat diet during pregnancy and lactation alters hepatic expression of insulin like growth factor-2 and key microRNAs in the adult offspring. <i> BMC Genomics</i>, 10:478, 2009.<br><a href="http://dx.doi.org/10.1186/1471-2164-10-478">Download</a> 
</li>
</ul>
<h3>
<a name="2008"></a>2008
</h3>
<ul>
<li>
X Didelot, J Taylor, and J Watkins. A Duality Identity between a Model of Bacterial Recombination and the Wright-Fisher Diffusion. <i> Markov Process. Relat. Top.</i>, 4:315–324, 2008.<br><a href="http://dx.doi.org/10.1214/074921708000000453">Download</a> 
</li>
<li>
X Didelot and D Falush. Bacterial Recombination in vivo In M Hensel and H Schmidt, editors, <i> Horiz. Gene Transf. Evol. Pathog.</i>, pp. 23–46, Cambridge University Press, 2008.<br><a href="http://www.cambridge.org/9780521862974">Download</a> 
</li>
<li>
S Kulick, C Moccia, X Didelot, D Falush, C Kraft, and S Suerbaum. Mosaic DNA imports with interspersions of recipient sequence after natural transformation of Helicobacter pylori. <i> PLoS One</i>, 3(11):e3797, 2008.<br><a href="http://dx.doi.org/10.1371/journal.pone.0003797">Download</a> 
</li>
<li>
H den Bakker, X Didelot, E Fortes, K Nightingale, and M Wiedmann. Lineage specific recombination rates and microevolution in Listeria monocytogenes. <i> BMC Evol. Biol.</i>, 8:277, 2008.<br><a href="http://dx.doi.org/10.1186/1471-2148-8-277">Download</a> 
</li>
</ul>
<h3>
<a name="2007"></a>2007
</h3>
<ul>
<li>
X Didelot, M Achtman, J Parkhill, N Thomson, and D Falush. A bimodal pattern of relatedness between the Salmonella Paratyphi A and Typhi genomes: Convergence or divergence by homologous recombination?. <i> Genome Res.</i>, 17(1):61–8, 2007.<br><a href="http://dx.doi.org/10.1101/gr.5512906.1">Download</a> 
</li>
<li>
X Didelot and D Falush. Inference of bacterial microevolution using multilocus sequence data. <i> Genetics</i>, 175(3):1251–66, 2007.<br><a href="http://dx.doi.org/10.1534/genetics.106.063305">Download</a> 
</li>
</ul>
<h3>
<a name="2006"></a>2006
</h3>
<ul>
<li>
D Falush, M Torpdahl, X Didelot, D Conrad, D Wilson, and M Achtman. Mismatch induced speciation in Salmonella: model and data. <i> Philos. Trans. R. Soc. London</i>, 361(1475):2045–53, 2006.<br><a href="http://dx.doi.org/10.1098/rstb.2006.1925">Download</a> 
</li>
</ul>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
